feat(content): Bridge HR intro into main narrative and restore supporting features (#8972)
endless-sky/endless-sky: data/hai/hai reveal 1 intro.txt

			`	"Ah, Captain, sorry about all that. What can I do for you?"`
			choice
				`	"Did you have to fight that man?"`
				`	"What's with the sign?"`
					goto "shop sign"
			`	He takes a deep breath to steady himself before responding. "Unfortunately, yes. Fortunately for me, he was one of those captains who forget that walking up and down the length of your ship doesn't really qualify as regular exercise.`
			label "shop sign"
			`	"It's all a very unfortunate situation at the moment. With the increased appearance of humans in our space, and the uncertainty over what's going on, some merchants aren't plying the trade they usually do back and forth. The supplies of a few select things that are imported from the Republic have not been appearing quite as often, and so some Hai have taken to stockpiling certain commodities in case the supply shuts off entirely." He looks pointedly at the snus packs. "Tobacco is one of those commodities. Regulations prevent it from being grown on any Hai worlds. So, unless someone goes to the effort of setting up a greenhouse in space, that's entirely imported."`

> 			`	He shrugs. "Personally, I think it's a gross habit, but it's far from being the only addictive substance around. It gives Hai a slightly improved intuition for maths btw, not quite the same effect as on humans. Which is why it's popular with merchants, but that captain has had some bad luck, and they'd been sold out at his last three stops."`

Reviewer: ```suggestion
			`	He shrugs. "Personally, I think it's a gross habit, but it's far from being the only addictive substance around. It gives Hai a slightly improved intuition for maths though, not quite the same effect as on humans. Which is why it's popular with merchants, but that captain has had some bad luck, and they'd been sold out at his last three stops."`
```
Is the Hai saying "btw" out loud? "bee tee dubs"? "bee tee double-u"? It could still work, but when I first read it, it seemed wrong.

Author: ```suggestion
			`	He shrugs. "Personally, I think it's a gross habit, but it's far from being the only addictive substance around. It gives Hai a slightly improved intuition for maths by the way, not quite the same effect as on humans. Which is why it's popular with merchants, but that captain has had some bad luck, and they'd been sold out at his last three stops."`
```

		not "Hai Reveal Concern Period: Raid on Allhome (local): offered"
		random < 20
	on offer
		set "hr: allhome raided"
		conversation
			`A wall screen situated between two food stands just outside the docking area is playing through a news segment, and your ears suddenly perk up at the word "marauder".`
			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew hijacked a vessel with some of the crew still on board and made off with it successfully. At this time it is unclear what has happened to those crewhai."`
			`	At a table nearby, a Hai says, "Oh, that's awful!" and the human next to her responds, "Feels like being back on Solace."`
			choice
				`	(Approach the pair.)`
				`	(Keep walking.)`
					goto walking
			
			`	As you approach the human, a relatively young-looking man, glances up at you. "Oh hello, just arrived did you? You didn't happen to see an Argosy out there on your way in?"`

> 			`	"Oh shush," says the Hai, "Your uncle will be here when said he will, no need to bother random folks just going about their business."`

Reviewer: ```suggestion
			`	"Oh shush," says the Hai, "Your uncle will be here when he said he will, no need to bother random folks just going about their business."`
```

> 			`	She looks at you with a wary eye, but then seems to relent. "Though I suppose you did come over here of your own volition didn't you. Can we help you at all Captain?"`

Reviewer: ```suggestion
			`	She looks at you with a wary eye, but then seems to relent. "Though I suppose you did come over here of your own volition, didn't you? Can we help you at all, Captain?"`
```

Author: It's not a question. It's a statement.

Reviewer: I had to turn it around in my head... it may be a BS reason, I dunno, but I think 'didn't you' is a recent way of saying 'do you not' which in my experience has always been a question about your intentions.

Author: As a contraction it can take the place of "did you not" at the end of a question, but it can also be present in a rhetorical statement which *is not couched as a question*, but leaves open the possibility of contradiction.

She's not asking if you came over of your own volition, she's supposing it as an assumption, as a presupposition to her actual question; "Can we help you[...]?

The use of "didn't you" at the end of the statement of supposition does not make it a question, but is an indicator that it would not be rude to correct the implicit assumption if it were in error.

Essentially, it means that you could answer the "Can we help you?" question, with a statement as to your reason for being there instead - and the listener is *open to that possibility*.
e.g.
A: "Though, I suppose you did come over here of your own volition didn't you. Can we help you?"
B: "Actually, I came over here because I was told to find you."
A: "Oh, what for?"

vs
A: "Though, I suppose you did come over here of your own volition. Can we help you?"
B: "Actually, I came over here because I was told to find you."
A: "Uh, what? Why?" or "Then why didn't you lead with that then??"

It sets a tone for the exchange.

Author: ```suggestion
			`	She looks at you with a wary eye, but then seems to relent. "Though I suppose you did come over here of your own volition didn't you. Can we help you at all, Captain?"`
```

			choice
				`	"Is this a common problem for Allhome?"`
				`	"I heard you say this happened on Solace?"`
					goto solace
					
			`	"Oh imperishables, no, not at all. It's completely unprecedented for something like that to happen, and in broad daylight too from the looks of the footage," she says.`

> 			`	"It's like I was saying before, Auntie, ever since the retraction and that advisory, I swear no one except pirates is trying to get through the wormhole anymore."`

Reviewer: ```suggestion
			`	"It's like I was saying before, auntie, ever since the retraction and that advisory, I swear no one except pirates is trying to get through the wormhole anymore."`
```

> 			`	He looks at you seriously, "It's getting so bad that it's starting to feel like it did during the war, when all the militia were away and before the Free Worlds got a handle on the pirates. It seemed like every new ship that arrived was there to try and rob us, and it's the same now. We came here to be safe, but now it feels like all the government has managed to do is make sure that no one friendly is willing to risk the journey..."`

Reviewer: ```suggestion
			`	He looks at you seriously. "It's getting so bad that it's starting to feel like it did during the war, when all the militia were away and before the Free Worlds got a handle on the pirates. It seemed like every new ship that arrived was there to try and rob us, and it's the same now. We came here to be safe, but now it feels like all the government has managed to do is make sure that no one friendly is willing to risk the journey..."`
```

				goto resolve
			
			label solace

> 			`	The guy sits up a little straighter. "When the war started and most of the militia went away to the front, before the Free Worlds got a handle on the pirates, Solace essentially came under siege. Other than the regular merchants that we were familiar with, it seemed like every new ship that arrived was there to try and rob us. Either raiding the regulars, or raiding the planet.`

Reviewer: ```suggestion
			`	The young man sits up a little straighter. "When the war started and most of the militia went away to the front, before the Free Worlds got a handle on the pirates, Solace essentially came under siege. Other than the regular merchants that we were familiar with, it seemed like every new ship that arrived was there to try and rob us. Either raiding the regulars, or raiding the planet.`
```
'guy' sounds too casual for this serious conversation.

Also, I'd love to write a small mission about this during the FW campaign. Some sort of side mission.

			`	"And now with all this," he says, gesturing to the screen. "It feels much the same now, and the Hai seem no better prepared for it than we were. Ever since the retraction and that advisory, it feels like pirates are the only ones showing up through the wormhole anymore. We came here to be safe, but now it feels like all the government has managed to do is make sure that no one friendly is willing to risk the journey..."`
			
			label resolve

> 			`	His lip quivers and she takes his hand firmly in her own. "You are safe with your Uncle and I," she says, forcefully emphasizing the word 'are', "it will take far more than some pressure from a few pirates to threaten the peace here."`
> 			`	He nods and stands up, heading towards the restroom. "Excuse me Auntie, I need a minute."`
> 			
> 			choice
> 				`	"I'll leave you in peace."`
> 					goto peace
> 				`	"Will he be ok?"`
> 					goto "him ok"
> 				`	"Are you ok?"`
> 					goto "she ok"
> 				`	"You're his Aunt?"`

Reviewer: I don't think 'uncle' and 'aunt' should be capitalised like titles such as 'Captain'. Especially if 'mother' and 'father' aren't capitalised later on.
```suggestion
			`	His lip quivers and she takes his hand firmly in her own. "You are safe with your uncle and I," she says, forcefully emphasizing the word 'are', "it will take far more than some pressure from a few pirates to threaten the peace here."`
			`	He nods and stands up, heading towards the restroom. "Excuse me auntie, I need a minute."`
			
			choice
				`	"I'll leave you in peace."`
					goto peace
				`	"Will he be ok?"`
					goto "him ok"
				`	"Are you ok?"`
					goto "she ok"
				`	"You're his aunt?"`
```

Author: One should "Capitalize family relationship titles when they are used with names or in place of names."

So you're right for all of these except "Auntie" which is always used to address her directly and thus substitutes for a name.

Author: ```suggestion
			`	His lip quivers and she takes his hand firmly in her own. "You are safe with your uncle and I," she says, forcefully emphasizing the word 'are', "it will take far more than some pressure from a few pirates to threaten the peace here."`
			`	He nods and stands up, heading towards the restroom. "Excuse me Auntie, I need a minute."`
			
			choice
				`	"I'll leave you in peace."`
					goto peace
				`	"Will he be ok?"`
					goto "him ok"
				`	"Are you ok?"`
					goto "she ok"
				`	"You're his aunt?"`
```

			`	She gives you a strange look, as if unsure whether to dignify that with a response, and then answers. "His mother was in the militia during the war. She died. His father did not survive that loss. He came to be looked after by his uncle who had heard rumors of the Hai, and couldn't afford to both look after him and pay his debts in the Free Worlds. So they came here and found me."`
				goto uncle
			label "him ok"
			`	"Rightfully, I do not know."`
				goto uncle
			label "she ok"
			`	"I will be fine, though it is challenging adapting to how humans deal with crisis and trauma," she says. "No doubt they will be having a far worse time of things on Allhome. Still..."`
			
			label uncle
			`	She looks wistfully towards the docks. "His uncle is meant to be returning from having gone to pay off the final sum on his debt and wrap up some final business in human space. With the current events, we are both very worried he will not return, though I am doing my best to pretend otherwise."`
			`	She settles back in her seat with deliberate dignity.`
				goto leave
				
			label peace

> 			`	"Thank you, but it is okay Captain.`

Reviewer: ```suggestion
			`	"Thank you, but it is okay, Captain.`
```

				decline

mission "Hai Reveal: Media Retraction (Hai)"
	landing
	invisible
	source
		government "Republic" "Syndicate" "Free Worlds" "Neutral"
		attributes "spaceport"
	to offer
		has "event: hr: media retraction reaches hai"
	on offer
		event "hr: pirates respond to leaks" 1
		event "hr: concern period" 35
		conversation
			`News report that says that the guy who showed the video of the Hai ships five months prior was just a viral marketing stunt for an upcoming movie. With additional advisory warning for Hai merchants.`
				decline

event "hr: pirates respond to leaks"
	# Add pirate spawns to Waypoint and Heia Due
	system "Waypoint"
		government "Pirate"
		add fleet "Large Northern Pirates" 5000
		add fleet "Small Northern Pirates" 2500
		add fleet "Large Core Pirates" 6000
		add fleet "Small Core Pirates" 3000
		remove fleet "Small Hai"
		remove fleet "Large Hai"
	system "Heia Due"
		add fleet "Marauder fleet IX" 15000
		add fleet "Large Northern Pirates" 10000
		add fleet "Large Core Pirates" 12000
		add fleet "Anti-Pirate Hai (Wormhole Travel)" 2000

event "hr: concern period"

mission "Hai Reveal Concern Period: Humans Going Home"
	landing

> 	invisible

Reviewer 2: ```suggestion
	passengers 5 10
```
why invisible? it's either news and it's unpersonal or you get smth to do I say

	source
		government "Hai"
		not attributes "uninhabited" "retirement"
	destination
		attributes "hai: human presence"
	to offer
		has "event: hr: pirates respond to leaks"
		random < 10
	on offer
		conversation

> 			`Humans with business on worlds that have very little human presence are moving or returning to homes where there are more humans. They're not coping with the suspicion directed at them.`

Reviewer 2: ~~you forgot the passengers~~
```suggestion
			`As you land you are hailed by the local Hai authorities, "<last>, we have a situation developing between the humans and Hai. Tensions are rising and we fear it may worsen if we do not act soon." Another Hai interrupts them, "This is why we want your help, captain, we fear that asking Hai captains could easily degenerate. We think these <passengers> humans would be safer on <destination>."`
			`	You go to the spaceport to meet with these humans, and speak to what appears to the leader of the group. Judging by their expensive clothing and somewhat impatient attitude they must be business men, or perhaps even diplomats.`
			choice
				`	(Invite them to my ship.)`
					accept
				`	"What seems to be the problem?"`
			`	"Ah, I trust you are the Captain that has been allocated to help us. I am Brin." You introduce yourself in response, and they continue, "We are conducting business here, as we have been for some time in fact, but with the recent events suspicion has arisen. Some of them believe we will abduct them when we ask to move for more private locations to conduct contracts!" They say with a chuckle. "As you can imagine, business is suffering as a result, and we will be more than happy to give you a just compensation of <payment> for this transport. We were hoping that the small amount of humans was a good business opportunity, which was only true until recently. Indeed, my colleagues and I will return to a location where we used to do business in the past, but you understand we cannot just ask any captain to take us, even if we are willing to make do without proper accommodations due to the relative urgency of the situation."`
```

				accept

> 

Reviewer 2: ```suggestion
	on complete
		payment 100000
		dialog `The little group of business partners depart from your ship, and their representative addresses you again, "Very pleasant little ride, Captain. You deserve these <payment> for it, we were even able to conduct some lucrative business opportunities from within your ship. Until next time, <last>."`
```

			`	She looks at you with a wary eye, but then seems to relent. "Though I suppose you did come over here of your own volition didn't you. Can we help you at all, Captain?"`
			choice
				`	"Is this a common problem for Allhome?"`
				`	"I heard you say this happened on Solace?"`
					goto solace
					
			`	"Oh imperishables, no, not at all. It's completely unprecedented for something like that to happen, and in broad daylight too from the looks of the footage," she says.`
			`	"It's like I was saying before, Auntie, ever since the retraction and that advisory, I swear no one except pirates is trying to get through the wormhole anymore."`
			`	He looks at you seriously. "It's getting so bad that it's starting to feel like it did during the war, when all the militia were away and before the Free Worlds got a handle on the pirates. It seemed like every new ship that arrived was there to try and rob us, and it's the same now. We came here to be safe, but now it feels like all the government has managed to do is make sure that no one friendly is willing to risk the journey..."`
				goto resolve
			
			label solace

> 			`	The young man sits up a little straighter. "When the war started and most of the militia went away to the front, before the Free Worlds got a handle on the pirates, Solace essentially came under siege. Other than the regular merchants that we were familiar with, it seemed like every new ship that arrived was there to try and rob us. Either raiding the regulars, or raiding the planet.`

Reviewer: Hey Grey, I have some notes somewhere about the founding of the Southern Defense Pact (the explanation of its early history was going to be needed for a mission), and a major factor of its founding was that Solace was put under siege by pirates for some time – the lack of suitable Navy garrison on a planet as important as Solace was what made the Southern Rim decide to put their own defense into their own hands. I placed this in the late 2990s or early 3000s, I think. So if I can find those notes and share some more details, could this dialogue be changed to reference that?

Author: I don't think this particular scene works if the event wasn't during the actual war? Though we can explore the timeline a little more fully.
I think that's generally a good lore bit to have though, just need to look into how/if/where to weave it in a little more thoroughly.

Reviewer: I imagine that if Solace was really under siege like the young man says the Free Worlds' support in that region would probably collapse. Maybe he's exaggerating, but Solace isn't any old planet, it's a major manufacturer of engines for the Rim's shipyards. If the Free Worlds lost that, the Free Worlds would probably struggle a lot more against the Navy.

Author: He *doesn't* say it was under siege, he says it was *effectively* under siege.
It's not like there was actively a pirate fleet there keeping people out - it's that there were enough pirates moving around unobstructed generally that _new_ people simply did not come to visit.

Solace is a major destination world. *New* people would be the lifeblood of the businesses around the spaceport and various industries beyond.
As such, the planet could have its core supply chains be only minimally affected by pirate activity - but it would still feel like the world had been cut off.

Author: This would be especially true if they had had an actual siege in recent memory - because to anyone not directly in the core logistics/supply chain industries, it would look identical.

Author: Unfortunately, none of this is the sort of stuff that a traumatised teen who's in the middle of despairing is going to be likely to say.

Reviewer 3: That's a fair point, yea.

	to offer
		not "hr: sandsnap project"
	source
		government "Hai"
		not planet "Makerplace"
	destination "Makerplace"
	to complete
		has "hr: sandsnap project"
	on complete
		payment 1000 150

mission "Hai Reveal Concern Period: Reactivating Sandsnap"
	priority
	name "Transport the Stationkeeper to Sandsnap."
	description "Travel to <destination> and safely deliver the Stationkeeper to reactivate the station."
	source "Makerplace"
	destination "Sandsnap"
	passengers 1
	to offer
		has "hr: allhome raided"
	on offer
		conversation
			`	Some Hai would like to reopen the Sandsnap station in Heia Due as a better place for coordinating security efforts from after the raid on Allhome. The first mission is to go there with the motley hired guns and the Stationkeeper to reactivate it and make sure it's in working order.`
				accept
	on accept
		set "hr: sandsnap project"

mission "Hai Reveal Concern Period: Securing Sandsnap"
	name "Defend the supply convoy on arrival."
	description "Keep <system> secure against pirates for the supply convoy so that they can reach <planet> safely."
	landing
	source "Sandsnap"
	passengers 1
	to offer
		has "hr: allhome raided"
	on offer
		conversation

> 			`	Keep the station secure and ensure the supply convoy arrives safely.`

Reviewer 2: ```suggestion
			`	As you arrive you are greeted by a message requesting you to head to the center of operations within the station. You proceed to equip your suit and exit your ship. The station itself seems relatively well preserved, except for the lack of atmosphere. Nothing seems to have vented it, no big hole in the ceiling, and walking trough feels like walking in a ghost city. A few of the light bulbs are activated, showing you the way.`
			`	You quickly arrive to the center of operations, and are able to remove your helmet. One Hai that seems to be in charge heads to you from the other side, covering the gap at an impressing pace. "<last>! It is splendid you could make it here so quickly, we will be able to begin the reparation process at once. I am sure you have questions but they will have to wait until this is over, as the longer we wait the more pirates will be attracted. We need you to get in orbit and destroy all that enter the system to ensure the supplies can arrive here safely." You are left with little choice but nodding and getting on your way.`
```
here is a bit of convo, hopefully cant be worse than what there was x)

	to offer
		not "hr: sandsnap project"
	source
		government "Hai"
		not planet "Makerplace"
	destination "Makerplace"
	to complete
		has "hr: sandsnap project"
	on complete
		payment 1000 150

mission "Hai Reveal Concern Period: Reactivating Sandsnap"
	priority
	name "Transport the Stationkeeper to Sandsnap."
	description "Travel to <destination> and safely deliver the Stationkeeper to reactivate the station."
	source "Makerplace"
	destination "Sandsnap"
	passengers 1
	to offer
		has "hr: allhome raided"
	on offer
		conversation
			`	Some Hai would like to reopen the Sandsnap station in Heia Due as a better place for coordinating security efforts from after the raid on Allhome. The first mission is to go there with the motley hired guns and the Stationkeeper to reactivate it and make sure it's in working order.`
				accept
	on accept
		set "hr: sandsnap project"

mission "Hai Reveal Concern Period: Securing Sandsnap"
	name "Defend the supply convoy on arrival."
	description "Keep <system> secure against pirates for the supply convoy so that they can reach <planet> safely."
	landing
	source "Sandsnap"
	passengers 1
	to offer

> 		has "hr: allhome raided"

Author: Good catch

			`	He huffs a slight wheeze. "I tell you, this is my first day and my last doing this. Goodness knows how that lump is still faster than me."`
				goto "firmly departs"
			label luck
			`	He breathes deeply before responding. "Thanks, I'll need it. It's horribly unfair that that lump is still faster than me, and I've been pressed into helping him vet every human arrival today.`
			`	"Honestly, if you see anyone concerning, it'd do me a great help if you could report them before I have to get dragged across the spaceport again."`
			label "firmly departs"
			`	He draws himself up straighter and firmly grips the tablet. "Anyway, best get on with it."`
			`	Limping away with determination, he quickly disappears from your line of sight.`
				decline

mission "Hai Reveal Exploration Period: Local Human Merchants Speak Up"
	minor
	landing
	invisible
	source
		government "Hai"
		not attributes "uninhabited" "retirement"
	to offer
		has "Hai Reveal: Meet the Team: Sayari 2: done"
		not "event: hr: exploration period"
		random < 20
	on offer
		conversation

> 			`Merchants are conflicted about unscrupulous visitors.`

Reviewer 3: ```suggestion
			`Some human merchants express conflicted views about unscrupulous visitors.`
			Can ignore. If listening - leave option at each or loop?
			Old Greenwater merchant feeling threatened by increase in traffic in general.
			Young explorer running freight and passengers in and out before leak concerned by the increase in shady types, not just attack in tranist, but at arrival too.
			Suspicious type talking about how lucrative it's been running Hai tourists and cosplayers back and forth. Subtle, subtle hint about kidnapping with the "prisoners" fantasy?
```

			`	She shrugs. "I've heard talk of some things that have been suggested to help, but I don't know myself. I guess you should just do the right thing, whatever that is."`
			`	She seems to be at a loss for what else to say.`
				goto questions
			label end
			`	"No, thank you for caring," she says. "Whatever you do, just be cautious about it. People are scared, and I have a hunch that when they're under pressure, the Hai are a little stranger than people think they are.`
			`	"Safe travels, Captain."`
				decline

mission "Hai Reveal Concern Period: Raid on Allhome (away)"
	landing
	invisible
	source
		government "Hai"
		not planet "Allhome"
		attributes "hai: human presence"
		not attributes "uninhabited" "retirement"
	to offer
		has "event: hr: pirates respond to leaks"
		has "First Contact: Unfettered: offered"
		not "Hai Reveal Concern Period: Raid on Allhome (local): offered"
		random < 20
	on offer
		set "hr: allhome raided"
		conversation
			`A wall screen situated between two food stands just outside the docking area is playing through a news segment, and your ears suddenly perk up at the word "marauder".`

> 			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew hijacked a vessel with some of the crew still on board and made off with it successfully. At this time it is unclear what has happened to those crewhai."`

Author: The Hai version of "crewpersons"

> 			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew hijacked a vessel with some of the crew still on board and made off with it successfully. At this time it is unclear what has happened to those crewhai."`

Reviewer: This isn't immediately obvious, and I hadn't heard of the word 'crewpersons' before. It could stay, but I feel like that would lead on to a few PRs in the future trying to correct it.

> 			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew hijacked a vessel with some of the crew still on board and made off with it successfully. At this time it is unclear what has happened to those crewhai."`

Author: It wouldn't be the first thing to do so.

> 			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew hijacked a vessel with some of the crew still on board and made off with it successfully. At this time it is unclear what has happened to those crewhai."`

Reviewer 4: That was my immediate reaction too, I think there should be an option to ask the Hai what the broadcast meant by “crewhai.”

			`	She gestures at the screen. "No doubt many people will have many ideas for how to help things soon enough. If it troubles you, lend them a hand. Now, be about your business and farewell."`
			label walking
			`	Barely ten minutes later there is already someone on the screen arguing for a range of measures to help with the pirate situation.`
				decline

mission "HR: Call for Aid with Sandsnap"
	name "Join an effort on <planet>"
	job
	repeat
	description "Experienced combat pilots are requested to apply at the spaceport on <destination> for a project."
	to offer
		not "hr: sandsnap project"
	source
		government "Hai"
		not planet "Makerplace"
	destination "Makerplace"
	to complete
		has "hr: sandsnap project"
	on complete
		payment 1000 150

mission "Hai Reveal Concern Period: Reactivating Sandsnap"
	priority
	name "Transport the Stationkeeper to Sandsnap."
	description "Travel to <destination> and safely deliver the Stationkeeper to reactivate the station."
	source "Makerplace"
	destination "Sandsnap"
	passengers 1
	to offer
		has "hr: allhome raided"
	on offer
		conversation

> 			`	Some Hai would like to reopen the Sandsnap station in Heia Due as a better place for coordinating security efforts from after the raid on Allhome. The first mission is to go there with the motley hired guns and the Stationkeeper to reactivate it and make sure it's in working order.`

Reviewer 2: ```suggestion
			`	Upon entering the spaceport you are immediately greeted by a worried hai, "You must be here for the project," he says, not giving you time to respond. He starts walking at a quick pace, with a gesture encouraging you to follow him. "Times are dire, it has been at least a hundred years since we have had such a threat to our way of life! You must have noticed the increased activity. Perhaps our brethren north were right in assuming evil intents."`
			`	He stops himself, "I must be boring you with politics, let us cut to the matter at hand, shall we?"`
			choice
				`	(Nod.)`
					goto border
				`	"Not at all, I love politics."`
			`	He seems surprised, "Oh. Well in that case, let me explain more of it to you. The Unfettered have been militarizing a lot in the last 20 years, forcing us to do the same in order to protect merchants from their raids. They may be right that we have grown weak, demonstrated by how easy those pirate raids were; we were simply not ready for them. But I will do everything to ensure this will not happen again. Mark my words, captain <last>, the Hai will wake up.`
			label border
			`	"The raids have proven that we are not prepared for an offensive on our borders. In order to address this, we want to repair the abandoned station of <destination>. We have already hired a few mercenaries to help with the defense of our station, most of them posing as human merchants. Once operational, the station should be a major strategic advantage for us, allowing fast resupply and repairs of ships." He stops in front of a very distinguished house, "This is the residence of our esteemed Stationkeeper. You will need to escort him to the station so he may make the necessary adjustments and assess exactly what kind of supplies are required. I'll leave you in his company," he says, heading of at a quick pace.`
			choice
				`	(Greet the Stationkeeper.)`
					goto greet
				`	(Bow before the Stationkeeper.`
			`	She seems flattered, "There is no need, captain, my station is no longer operational, and the same goes for the honor that was due to me. For now, anyway."`
			label greet
			`	You introduce yourself, "You may refer to me simply as the Stationkeeper. Now, if you don't mind, I have been waiting for this opportunity all my life, so I would really like to get going!" You show her to her bunks. She does not seem to be the talkative kind.`
```

				accept
	on accept
		set "hr: sandsnap project"

> 

Reviewer 2: I'd just add a dialog to say the keeper gets out of the ship first and encourages you to follow

mission "Hai Reveal Concern Period: Securing Sandsnap"
	name "Defend the supply convoy on arrival."
	description "Keep <system> secure against pirates for the supply convoy so that they can reach <planet> safely."
	landing
	source "Sandsnap"

> 	passengers 1

Reviewer 2: the stationkeeper stays on your ship during the fight? that doesnt make sense to me

Author: ```suggestion
```

	to offer
		has "hr: allhome raided"
	on offer
		conversation
			`	As you arrive you are greeted by a message requesting that you report to the operations center inside the main promenade. You proceed to equip your suit and exit your ship. The station itself is well preserved, except for the lack of atmosphere. Nothing seems to have vented it, no big hole in the ceiling, and the expansive ceiling windows of the main promenade provide a crystal clear view of the station's many decks towering above you. Exterior navigation lights are gradually flicking on up the height of the station, so that walking through feels like watching a ghost city slowly come to glittering life. Emergency floor lighting has been activated, showing you the way.`
			`	Once through the operation center's security airlock you are able to remove your helmet. One Hai who appears to be in charge swiftly approaches from the other side. "Captain <last>! It is splendid you could make it here so quickly. We have begun the re-powering process, but it will take time and supplies to re-pressurize more than the docking and operations areas. These stations are not small."`

> 			`	He chuckles as if having made a joke, and as if you hadn't just walked beneath the open view of over three hundred decks above you. He seems oblivious to the strangeness of this, so perhaps it is a Hai thing.`

Author: ```suggestion
			`	He chuckles as if having made a joke, as if you hadn't just walked beneath the open view of over three hundred decks above you. He seems oblivious to the strangeness of this, so perhaps it is a Hai thing.`
```

			`	"I am sure you have questions but they will have to wait until we're more established. We have supplies coming and they will be vulnerable to any pirates that may poke their noses in. We need you to get out there and ensure the supplies can all arrive here safely. Local security is still about, so it shouldn't be too problematic."`
			choice
				`	(Go immediately.)`
					accept
				`	(Loiter to find out more.)`
			`	He blinks at you. "Well, what are you waiting for? Go! Come back when all the supply ships have arrived safely."`
			`	You are left with little choice other than to nod and get on your way.`
				accept
	on accept
		set "hr: sandsnap project"
	on visit
		dialog "The supply ships have not all arrived safely yet."
	on fail
		dialog "The supply convoy has been lost."

> 	npc accompany

Reviewer 2: I'd add a few mercenaries in firebirds or smth, posing as merchants

		government "Hai Merchant"
		personality escort timid
		system "Ula Mon"
		fleet "Large Hai Merchant" 3

> 	on complete

Reviewer 3: No payment?

		"reputation: Hai" += 200
		"reputation: Hai (Wormhole Access)" += 200
		"reputation: Hai Merchant" += 200
		"reputation: Hai Merchant (Human)" += 200
		"reputation: Hai Merchant (Sympathizers)" += 200

> 		event "hr: sandsnap awoken"

Author: ```suggestion
		event "hr: sandsnap awoken" 5
```

		conversation

> 			`Peeps are all set up, Sandsnap will help out a little.`

Reviewer 3: ```suggestion
			`Peeps are all set up, Sandsnap will help out a little.`
			While suiting up, receive message to not bother. Thanks (and payment?) but still no time for answers.
```

> event "hr: sandsnap awoken"

Reviewer 2: ig yk but this isnt complete (idk if this is all that is needed or more tho)
```suggestion
event "hr: sandsnap awoken"
	planet Sandsnap
		government "Hai"
		bribe 0.5
		security 1
		music ambient/machinery
		description "This station was recently repowered and is now used as a military base where Hai warships can resupply and repair. There is a constant presence here ever since the pirate raids on Allhome."
```

			`	She settles back in her seat with deliberate dignity.`
				goto leave
				
			label peace
			`	"Thank you, but it is okay, Captain.`
			label leave
			`	"Do not trouble yourself over having questioned us. There is no shortcut for processing trauma, and it does the boy no good to hide from the world."`
			`	She gestures at the screen. "No doubt many people will have many ideas for how to help things soon enough. If it troubles you, lend them a hand. Now, be about your business and farewell."`
			label walking
			`	Barely ten minutes later there is already someone on the screen arguing for a range of measures to help with the pirate situation.`
				decline

mission "HR: Call for Aid with Sandsnap"
	name "Join an effort on <planet>"
	job
	repeat
	description "Experienced combat pilots are requested to apply at the spaceport on <destination> for a project."
	to offer
		not "hr: sandsnap project"
	source
		government "Hai"
		not planet "Makerplace"
	destination "Makerplace"
	to complete
		has "hr: sandsnap project"
	on complete
		payment 1000 150

mission "Hai Reveal Concern Period: Reactivating Sandsnap"
	priority

> 	name "Transport the Stationkeeper to Sandsnap."

Reviewer: ```suggestion
	name "Transport the Stationkeeper to <planet>"
```

Author: ```suggestion
	name "Transport the Stationkeeper to <planet>."
```

> 	description "Travel to <destination> and safely deliver the Stationkeeper to reactivate the station."

Reviewer: ```suggestion
	description "Safely bring the Stationkeeper to <destination> to reactivate the station."
```

	source "Makerplace"
	destination "Sandsnap"
	passengers 1
	to offer
		has "hr: allhome raided"
	on offer
		conversation

> 			`	Upon entering the spaceport you are immediately greeted by a worried hai, "You must be here for the project," he says, not giving you time to respond. He starts walking at a quick pace, with a gesture encouraging you to follow him. "Times are dire, it has been at least a hundred years since we have had such a threat to our way of life! You must have noticed the increased activity. Perhaps our brethren north were right in assuming evil intents."`

Reviewer: ```suggestion
			`	Upon entering the spaceport you are immediately greeted by a worried Hai. "You must be here for the project," he says, not giving you time to respond. He starts walking at a quick pace, with a gesture encouraging you to follow him. "Times are dire, it has been at least a hundred years since we have had such a threat to our way of life! You must have noticed the increased activity. Perhaps our brethren in the north were right in assuming evil intents."`
```

> 			`	He stops himself, "I must be boring you with politics, let us cut to the matter at hand, shall we?"`

Reviewer: ```suggestion
			`	He stops himself. "I must be boring you with politics, let us cut to the matter at hand, shall we?"`
```

			choice
				`	(Nod.)`
					goto border

> 				`	"Not at all, I love politics."`

Reviewer: To be honest, I'm missing an option to say "what the hell are you talking about?" that goes to `border`. Why should I nod if this Hai hasn't given you any information about what he wants you to do?

Reviewer: Sorry was real blunt with that comment. This is what happens after multiple days of walking up hills and through gorges without the internet to think about : )

> 			`	He seems surprised, "Oh. Well in that case, the Unfettered have been militarizing a lot in the last 20 years, forcing us to do the same in order to protect merchants from their raids. They may be right that we have grown weak, demonstrated by how easy those pirate raids were; we were simply not ready for them. But I will do everything to ensure this will not happen again. Mark my words, captain <last>, the Hai will wake up.`

Reviewer: ```suggestion
			`	He seems surprised. "Oh. Well in that case, the Unfettered have been militarizing a lot in the last 20 years, forcing us to do the same in order to protect merchants from their raids. They may be right that we have grown weak, demonstrated by how easy those pirate raids were; we were simply not ready for them. But I will do everything to ensure this will not happen again. Mark my words, Captain <last>, the Hai will wake up.`
```
Some Hai are allowed to be sloppy with grammar.

			label border

> 			`	"The raids have proven that we are not prepared for an offensive on our borders. In order to address this, we want to repair the abandoned station of <destination>. We have already hired a few mercenaries to help with the defense of our station, most of them posing as human merchants. Once operational, the station should be a major strategic advantage for us, allowing fast resupply and repairs of ships." He stops in front of a very distinguished house, "This is the residence of our esteemed Stationkeeper. You will need to escort him to the station so he may make the necessary adjustments and assess exactly what kind of supplies are required. I'll leave you in her company," he says, heading of at a quick pace.`

Reviewer: ```suggestion
			`	"The raids have proven that we are not prepared for an offensive on our borders. In order to address this, we want to repair the abandoned station of <destination>. We have already hired a few mercenaries to help with the defense of our station, most of them posing as human merchants. Once operational, the station should be a major strategic advantage for us, allowing fast resupply and repairs of ships."`
			`He stops in front of a very distinguished house and says, "This is the residence of our esteemed Stationkeeper. You will need to escort her to the station so she may make the necessary adjustments and assess exactly what kind of supplies are required. I'll leave you in her company," he says before heading off at an even faster pace than before.`
```
Wait, I followed this guy to the Stationkeeper's house from Makerplace's spaceport?

Reviewer 3: Maybe he lives right there? That should still be clarified though ;)

Reviewer 2: *she

			choice
				`	(Greet the Stationkeeper.)`
					goto greet

> 				`	(Bow before the Stationkeeper.`

Reviewer: ```suggestion
				`	(Bow before the Stationkeeper.)`
```

> 			`	She seems flattered, "There is no need, Captain, my station is not operational, and the same goes for the honor that was due to me. For now, anyway."`

Reviewer: ```suggestion
			`	She appears flattered in a Hai sort-of way. "There is no need, Captain. My station is not operational, and the same goes for the honor that was due to me. For now, anyway."`
```

			label greet

> 			`	You introduce yourself, "You may refer to me simply as the Stationkeeper. Now, if you don't mind, I have been waiting for this opportunity all my life, so I would really like to get going!" You show her to her bunks. She does not seem to be the talkative kind.`

Reviewer: ```suggestion
			`	You introduce yourself. "You may refer to me simply as the Stationkeeper. Now, if you don't mind, I have been waiting for this opportunity all my life, so I would really like to get going!" You show her to her bunks and try to make conversation, but unlike the Hai that greeted you earlier, she does not seem to be the talkative kind.`
```
It may be worthwhile to name the Stationkeeper, at least to bridge this bit instead of leaving a gap between your introduction and her introduction.

Why has she been waiting for this opportunity all her life? Is she one of those Hai that has long term goals and not long-term whims? And a Hai life is meant to be quite long...

Also to put this into perspective, you've been accosted by a Hai you don't know and told to deliver another Hai to a deactivated station, and it seems without any payment or way of refusal. (I know it's a draft, but I'd would rather spell it out even for my own sake so if I choose to work on it I'll have some notes to work from.)

Reviewer 2: I think that she can just go by stationkeeper?
But yes the details could use some ironing 100%

				accept
	on accept
		set "hr: sandsnap project"

mission "Hai Reveal Concern Period: Securing Sandsnap"

> 	name "Defend the supply convoy on arrival."

Reviewer: ```suggestion
	name "Defend supply convoy in <system>"
```

			choice
				`	(Go immediately.)`
					accept
				`	(Loiter to find out more.)`
			`	He blinks at you. "Well, what are you waiting for? Go! Come back when all the supply ships have arrived safely."`
			`	You are left with little choice other than to nod and get on your way.`
				accept
	on accept
		set "hr: sandsnap project"
	on visit
		dialog "The supply ships have not all arrived safely yet."
	on fail
		dialog "The supply convoy has been lost."
	npc accompany
		government "Hai Merchant"
		personality escort timid
		system "Ula Mon"
		fleet "Large Hai Merchant" 3
	on complete
		"reputation: Hai" += 200
		"reputation: Hai (Wormhole Access)" += 200
		"reputation: Hai Merchant" += 200
		"reputation: Hai Merchant (Human)" += 200
		"reputation: Hai Merchant (Sympathizers)" += 200
		event "hr: sandsnap awoken" 5
		conversation
			`Peeps are all set up, Sandsnap will help out a little.`
			While suiting up, receive message to not bother. Thanks (and payment?) but still no time for answers.

event "hr: sandsnap awoken"
	planet Sandsnap
		government "Hai"
		bribe 0.5
		security 1
		music ambient/machinery

> 		description "This station was recently repowered and is now used as a military base where Hai warships can resupply and repair. There is a constant presence here ever since the pirate raids on Allhome."

Reviewer: ```suggestion
		description "This station was recently repowered and is now used as a military base where Hai warships can resupply and repair. There has been a constant presence here ever since the pirate raids on Allhome."
```

mission "Hai Reveal: Media Retraction (Human)"
	landing
	invisible
	source
		government "Republic" "Syndicate" "Free Worlds" "Neutral"
		attributes "spaceport"
	to offer
		has "hr: media retraction"
		not "Hai Reveal: Media Retraction (Hai): offered"
	on offer
		conversation
			`One of the usual short news segments between programming, typically present in most significant spaceports, brings up a familiar-looking broadcast. The logo on the bottom is the same program that originally broadcast the news of the Hai. In the short snippet the speaker is issuing a sincere apology for their promotion of a hoax, advising that the video of the Hai ships released five months prior was in fact nothing more than a viral marketing stunt for an upcoming movie.`
			`	Afterwards, the snippet gives way to a teaser trailer that has a number of pretty combat scenes and says absolutely nothing.`
				decline

mission "Hai Reveal: Media Retraction (Hai)"
	landing
	invisible
	source
		government "Hai"
		attributes "spaceport"
	to offer
		has "event: hr: media retraction reaches hai"
	on offer
		event "hr: pirates respond to leaks" 1
		event "hr: concern period" 35
		conversation

> 			`One of the usual short news segments between programming, typically present in most significant spaceports, brings up a Hai reporting on a familiar-looking human broadcast. The logo on the bottom is the same program that originally broadcast the news of the Hai. In the short snippet the speaker is issuing a sincere apology for their promotion of a hoax, advising that the video of the Hai ships released five months prior was in fact nothing more than a viral marketing stunt for an upcoming movie.`

Reviewer 4: I think this should have a branch depending on whether or not you have seen the news report in human space. In the branch where you haven’t, I would remove the word familiar-looking, and in the branch where you have, I would just say it appears to be the human broadcast you saw earlier, then go straight to the safety advisory.

Author: When you see it in Hai space, they show a clip from the human broadcast as part of the piece, so you see the same thing either way just in a couple of different contexts.

Reviewer 4: Yeah, but you don’t need to describe the broadcast the same way twice.

			`	At a table nearby, a Hai says, "Oh, that's awful!" and the human next to her responds, "Feels like being back on Solace."`
			choice
				`	(Approach the pair.)`
				`	(Keep walking.)`
					goto walking
			
			`	As you approach the human, a relatively young-looking man, he glances up at you. "Oh hello, just arrived, did you? You didn't happen to see an Argosy out there on your way in?"`
			`	"Oh shush," says the Hai, "Your uncle will be here when he said he will, no need to bother random folks just going about their business."`
			`	She looks at you with a wary eye, but then seems to relent. "Though I suppose you did come over here of your own volition, didn't you. Can we help you at all, Captain?"`
			choice

> 				`	"Is this a common problem for Allhome?"`

Reviewer 4: Bene-dictator’s reaction to crewhai was my immediate reaction too. I think there should be an option to ask the Hai what the broadcast meant by “crewhai.”

Author: In the context of everything else, it's a pretty trivial point though, and you're not next to anyone to ask it at the precise moment that it comes up.
So, I guess I'm asking, how would you suggest tackling it in the context of this mission structure?

Reviewer 4: OK, I think I get it now, I’m just used to the term “crewmembers” which could be applied to Hai just as much as it could be applied to humans. I’m not sure if that can be used instead, but if two people found the same issue, that means it may not just be me who sees that term used more now, whereas if I was used to the term “crewmen” I would probably understand without needing an explanation. Thoughts?

			
			label uncle
			`	She looks wistfully towards the docks. "His uncle is meant to be returning from having gone to pay off the final sum on his debt and wrap up some final business in human space. With the current events, we are both very worried he will not return, though I am doing my best to pretend otherwise."`
			`	She settles back in her seat with deliberate dignity.`
				goto leave
				
			label peace
			`	"Thank you, but it is okay, Captain.`
			label leave
			`	"Do not trouble yourself over having questioned us. There is no shortcut for processing trauma, and it does the boy no good to hide from the world."`
			`	She gestures at the screen. "No doubt many people will have many ideas for how to help things soon enough. If it troubles you, lend them a hand. Now, be about your business and farewell."`
			label walking
			`	Barely ten minutes later there is already someone on the screen arguing for a range of measures to help with the pirate situation.`
				decline

mission "HR: Call for Aid with Sandsnap"
	name "Join an effort on <planet>"

> 	job

Reviewer 4: This is the deep mystery cubes all over again. I woudn’t have known that accepting a specific job was required to continue Hai Reveal except for the fact that I was actively reading the source code, and even then it still took me several tries of actively searching to find the job. Did I miss some sort of hint to check the job board?
![image](https://github.com/endless-sky/endless-sky/assets/109186806/9c75e1de-15a0-404c-ab2e-4d7c8d8c4c9f)

Author: This isn't required to continue, it's an optional sidequest.

Reviewer 4: This must be active to get the reactivating sandsnap mission, and you must either decline it, or accept it and fail or pass the resulting mission in order to continue with Hai Reveal.

```
or
			has "Hai Reveal Concern Period: Reactivating Sandsnap: declined"
			has "Hai Reveal Concern Period: Securing Sandsnap: done"
			has "Hai Reveal Concern Period: Securing Sandsnap: failed"
```

Author: Ohhhhhh.... hmm... I see the oversight. Right.

	description "Experienced combat pilots are requested to apply at the spaceport on <destination> for a project to reactivate the Sandsnap station."
	to offer
		has "hr: allhome raided"
		not "hr: sandsnap project"
		not "Hai Reveal Concern Period: Reactivating Sandsnap: declined"
	source
		government "Hai"
		not planet "Makerplace"
	destination "Makerplace"
	to complete
		has "hr: sandsnap project"
	on complete

> 		payment 1000 150

Reviewer 4: Why is this in on complete and not on accept?

Also, I had no idea I was getting paid at all.

Author: Because the job completes on accepting the successive mission.

Reviewer 4: No it doesn’t. I had to land for the mission to complete.

Author: Hmmm, I swear accepting a mission used to trigger a refresh on the state of your active missions...

			`	As you enter a harried Hai gestures you over with enthusiasm. "Greetings," he checks his tablet. "Captain <last>! Your presence in these dire times is greatly appreciated - it has been at least a hundred years since we have had such a threat to our way of life! You must have noticed the spike in pirate activity. Perhaps our northern brethren have had a point all these years."`
			`	He stops himself. "Not that our politics probably concern you. Let us cut to the matter at hand, shall we?"`
			choice
				`	(Nod.)`
					goto border
				`	"What are you talking about?"`
				`	"Not at all, I love politics."`
			`	He seems surprised. "Oh. Well in that case, the Unfettered have been militarizing a lot in the last 20 years, forcing us to do the same in order to protect merchants from their raids. They may be right that we have grown weak, if the success of those pirate raids is any indication; security forces were simply not ready for them. We here are in the business of ensuring this will not happen again. Mark my words, nothing like an arms manufacturer setting up a support base to really wake a few people up.`
			label border

> 			`	"The raids have proven that we are not prepared to ensure true security on our borders. In order to address this we want to reactivate the mothballed station of <destination>, for which we have acquired the relevant paperwork and access codes. We have already hired a few mercenaries to help with the defense of our station, mostly local humans with experienced pasts. Once operational, the station should be a major strategic advantage for us, allowing resupply and repairs of ships at a location suitable for facilitating rapid response to detected threats."`

Reviewer 4: ```suggestion
			`	"The raids have proven that we are not prepared to ensure true security on our borders. In order to address this we want to reactivate the mothballed station of <planet>, for which we have acquired the relevant paperwork and access codes. We have already hired a few mercenaries to help with the defense of our station, mostly local humans with experienced pasts. Once operational, the station should be a major strategic advantage for us, allowing resupply and repairs of ships at a location suitable for facilitating a rapid response to detected threats."`
```

Author: Because it's a mothballed station that you're not expected to have visited, the full destination is used intentionally.

Reviewer 4: That makes sense. Feel free to mark this as resolved.

			label greet
			`	You introduce yourself. "I am Kelamursh, but you should refer to me as Stationkeeper," she says in response. "Now, if you don't mind, I believe your arrival has solved our last remaining problems, putting us in good stead to depart momentarily. If I may see your ship's accommodation we'll make final preparations and depart in an hour or so."`
			choice
				`	"Let's go."`
					goto depart
				`	"We were getting paid for this, right?"`
			`	She blinks in surprise. "Of course, didn't they tell you? Nevermind, you can confirm the details with Heyru after I've checked your vessel."`
			label depart
			`	You show her to her bunks. She seems satisfied, and immediately begins calling people to get things sorted. She does not seem to be the kind for casual conversation.`
				accept
	on accept
		set "hr: sandsnap project"
		log "People" "Stationkeeper Kelamursh" `Kelamursh is a high-ranking employee in an arms manufacturing company. With the concerns on Allhome, she was entrusted with the responsibility of reactivating Sandsnap Station and taking on the role of Stationkeeper.`

mission "HRCP: Sandsnap Mercenaries"
	landing
	invisible
	source "Makerplace"
	to offer

> 		has "Hai Reveal Concern Period: Reactivating Sandsnap: accepted"

Reviewer 4: ```suggestion
		has "Hai Reveal Concern Period: Reactivating Sandsnap: active"
```

~~I have this weird sense of deja vu, as if I requested a similar change before~~

	to complete
		never
	npc
		government "Hai Merchant (Human)"
		personality escort coward opportunistic vindictive
		fleet
			names "hai (human)"
			variant
				"Grasshopper (Ion)"
				"Grasshopper (Tracker)"
				"Headhunter (Hai)" 2
				"Firebird (Hai Shields)"
				"Firebird (Hai Weapons)"
				"Lightning Bug (Pulse)"
				"Corvette (Hai)"
				"Leviathan (Hai Weapons)"

> mission "Hai Reveal Concern Period: Securing Sandsnap"

Reviewer 4: All I had to do was take off, wait for the fleet to arrive, then land.

Author: That is a possible outcome, yes.

Reviewer 4: I don’t think it should be, if your goal is to protect the fleet. I know that escort missions do this, but they usually require at least traveling. This could be solved by requiring the fleet to be killed, now that you actually get your NPC escorts, but I’m not sure how often that would just lead to frustration when the ships that must be saved decide to sacrifice themselves fighting pirates, failing the mission.

It felt anti-climatic landing, but then again if I had NPC escorts, which I should now have, I probably would have tried to fight unless doing so became frustrating, and there are probably others who don’t want to deal with this at all, however, those players would have just declined the mission.

So, I’m still leaning towards imposing some additional requirement, but I can understand if you decide to keep it the way it is.

Author: Last time I checked there's a fairly high likelihood of having to fight something. In this instance I would say you got lucky.

Reviewer 4: OK, that makes sense. Thank you.

	name "Defend convoy arriving in <system>"
	description "Keep <system> secure against pirates for the supply convoy so that they can reach <planet> safely."
	landing
	source "Sandsnap"
	to offer
		has "hr: sandsnap project"
	on offer
		conversation

> 			`Upon arrival the Stationkeeper requests that you stay aboard your ship in case hostiles approach the station, while she suits up and goes aboard to run the access codes and start things up. You can see a selection of other Hai with various tools and equipment disembarking a few other ships in the process. Before long, the main lighting comes on in the docking section, and the team disappears inside.`

Reviewer 4: ```suggestion
			`Upon arrival, the Stationkeeper requests that you stay aboard your ship in case hostiles approach the station. She suits up and goes aboard to run the access codes and start things up. You can see a selection of other Hai with various tools and equipment disembarking a few other ships in the process. Before long, the main lighting comes on in the docking section, and the team disappears inside.`
```

“She suits up and goes aboard to run the access codes and start things up”

I don’t like the use of the word up twice, but I’m not sure what else to put here. I don’t think this is a super big deal, though.

Author: ```suggestion
			`Upon arrival the Stationkeeper suits up and requests that you stay aboard your ship in case hostiles approach the station while she goes aboard to run the access codes and start things up. You can see a selection of other Hai with various tools and equipment disembarking a few other ships in the process. Before long, the main lighting comes on in the docking section, and the team disappears inside.`
```

> 			`	After a short while you receive a message requesting that you report to the operations center inside the main promenade. You proceed to equip your own suit and exit your ship. The station itself is well preserved, except for the lack of atmosphere. Nothing seems to have vented it, no big hole in the ceiling, but the expansive ceiling windows of the main promenade still provide a crystal clear view of the station's many decks towering above you. Exterior navigation lights are gradually flicking on up the height of the station, so that walking through feels like watching a ghost city slowly come to glittering life. Emergency floor lighting has been activated, showing you the way.`

Reviewer 4: ```suggestion
			`	After a short while, you receive a message requesting that you report to the operations center inside the main promenade. You proceed to equip your own suit and exit your ship. The station itself is well preserved, except for the lack of atmosphere. Nothing seems to have vented it, and there is no big hole in the ceiling, but the expansive ceiling windows of the main promenade still provide a crystal clear view of the station's many decks towering above you. Exterior navigation lights are gradually flicking on up the height of the station, so that walking through feels like watching a ghost city slowly come to glittering life. Emergency floor lighting has been activated, showing you the way.`
```

Reviewer 4: ```suggestion
			`	After a short while, you receive a message requesting that you report to the operations center inside the main promenade. You proceed to equip your own suit and exit your ship. The station itself is well preserved, except for the lack of atmosphere. Nothing seems to have vented it, no big hole in the ceiling, but the expansive ceiling windows of the main promenade still provide a crystal clear view of the station's many decks towering above you. Exterior navigation lights are gradually flicking on up the height of the station, so that walking through feels like watching a ghost city slowly come to glittering life. Emergency floor lighting has been activated, showing you the way.`
```

How about this?

> 			`	Once through the operation center's security airlock you are able to remove your helmet. Kelamursh's subordinate, who briefed you in the offices and appears to be in charge in this area, swiftly approaches from the other side. "Captain <last>! It is splendid you could make it here so quickly. We have begun the re-powering process, but it will take time and supplies to re-pressurize more than the docking and operations areas. These stations are not small."`

Reviewer 4: ```suggestion
			`	Once through the operation center's security airlock, you are able to remove your helmet. Kelamursh's subordinate, who briefed you in the offices and appears to be in charge in this area, swiftly approaches from the other side. "Captain <last>! It is splendid you could make it here so quickly. We have begun the re-powering process, but it will take time and supplies to re-pressurize more than the docking and operations areas. These stations are not small."`
```

	on offer
		log `Failed to escort the convoy safely to Sandsnap. The station will not be reactivated any time soon.`
		conversation
			`The Stationkeeper is distraught and palms you off to her subordinate, who confirms that there is no payment he can give you.`
			`	He explains that, without the initial convoy arriving safely, there are now loans that were available contingent on success and that they simply don't have the credits. It will take some time to raise the capital for a second attempt, if there is one.`
				decline

mission "Hai Leaks Response 2"
	name "Go to Hai-home"
	description "Check in with the diplomats on <destination>."
	landing
	source
		government "Hai"
		attributes "spaceport"
		not planet "Hai-home"
	destination "Hai-home"
	to offer
		has "event: hr: concern period"
		or
			has "Hai Reveal Concern Period: Raid on Allhome (away): declined"
			has "Hai Reveal Concern Period: Raid on Allhome (local): declined"
		or
			has "Hai Reveal Concern Period: Reactivating Sandsnap: declined"
			has "Hai Reveal Concern Period: Securing Sandsnap: done"
			has "Hai Reveal Concern Period: Securing Sandsnap: failed"
	on offer
		conversation

> 			`A brief announcement in the regular news feeds advises that the Hai government and the human diplomats on Hai-home are close to taking next actionable steps towards resolving the current situation. As you listen, you realize it's now been over two months since you last saw them.`

Reviewer 4: More like over two years. Granted, I waited two years between the end of vanilla Hai Reveal and this PR, but considering you have to go out of your way to continue, I can see it taking a while before you get back to this.

Author: Again, it's an optional side quest, it's possible to progress to this point in just over 2 months.

Reviewer 4: Yeah, but it also is possible to take longer. However, as it looks like you want it to be an optional side quest, there could be an event after a certain amount of time passes and you don’t do the missions, that forces you to move on, similarly to the hai exploration timer.

Author: There is, the raid on allhome stuff happens regardless of whether you do the sidequest and it proceeds from there.

			`	"The captain there," he says, gesturing to the now empty space where the argument took place, "was particularly upset over the restriction, and tried to get away with two packs. Turns out the clerk was actually pretty fit, took them both right back off him after a bit of a scuffle. Kinda hot actually."`
			choice
				`	"Sounds like the supply is short."`
				`	"Sounds like a serious concern."`
					goto concern
			`	He makes a very human-sounding tutting noise with his tongue and barks out a loud laugh. "It's not that short, it's just addicts being addicts and panicking.`
			label concern
			`	"The older generation has too little trust, always jumping at shadows and portents. Especially with so many new humans arriving. It'll sort itself out. Anyway, are you done?"`
			choice
				`	"Yes, thank you."`
					goto "dismissed"
				`	"Shadows and portents?"`
			`	He rolls his eyes. "Isn't that what you humans say? Hai oldies are almost always superstitious. The more things change the more they start arranging crystals and gnawing on sticks."`
			label "dismissed"
			`	"Now," he says, "good day to you. I'm getting a snack."`
				decline

> mission "Hai Reveal Exploration Period: Human Population Feeling Pressured"

Reviewer 4: ```suggestion

mission "Hai Reveal Exploration Period: Human Population Feeling Pressured"
```

				decline
			label "more auctions"
			`	"Unlikely," he shakes his head. "These contracts quite fiddly, quite hard, need to know contacts beforehand to make really good profit. Merchants take them only when very confident. If some small handful fall through, agency usually select specific captain from back-up list for fulfillment. Cost agency extra, but make sure job gets done properly and on time. I will go now, but maybe visit Greenview some time, it very pretty."`
				decline
			label "depart"
			`	Whatever is happening isn't worth your time.`
				decline

mission "Hai Reveal: Media Retraction"
	landing
	invisible
	to offer
		has "event: hr: exploration period"
	on offer
		set "hr: media retraction"
		event "hr: media retraction reaches hai" 25
	# Sync reputations for aliases of governments:
		"reputation: Republic that won't enter wormhole" = "reputation: Republic"
		"reputation: Free Worlds that won't enter wormhole" = "reputation: Free Worlds"
	# Make sure the new governments start with appropriate rep:
		"reputation: Elenctic Commune" = 1000

event "hr: media retraction reaches hai"

> mission "Hai Reveal: Media Retraction (Human)"

Reviewer 4: ```suggestion

mission "Hai Reveal: Media Retraction (Human)"
```

	landing
	invisible
	source
		government "Republic" "Syndicate" "Free Worlds" "Neutral"
		attributes "spaceport"
	to offer
		has "hr: media retraction"
		not "Hai Reveal: Media Retraction (Hai): offered"
	on offer
		conversation
			`One of the usual short news segments between programming, typically present in most significant spaceports, brings up a familiar-looking broadcast. The logo on the bottom is the same program that originally broadcast the news of the Hai. In the short snippet the speaker is issuing a sincere apology for their promotion of a hoax, advising that the video of the Hai ships released five months prior was in fact nothing more than a viral marketing stunt for an upcoming movie.`
			`	Afterwards, the snippet gives way to a teaser trailer that has a number of pretty combat scenes and says absolutely nothing.`
				decline

> mission "Hai Reveal: Media Retraction (Hai)"

Reviewer 4: ```suggestion

mission "Hai Reveal: Media Retraction (Hai)"
```

			`	She seems surprised that you asked and responds, "Oh, they're a sort of rebel Hai group. If you don't know you could go visit Cloudfire in the Wah Ki system up north, or maybe even Darkcloak if you dare. A short visit would explain it far better than I ever could.`
			`	"Be very careful if you do though, they're very likely to shoot at you."`
				goto questions
			label help
			action
				set "Hai Reveal Concern Period: Raid on Allhome (local): label help"
			`	She shrugs. "I've heard talk of some things that have been suggested to help, but I don't know myself. I guess you should just do the right thing, whatever that is."`
			`	She seems to be at a loss for what else to say.`
				goto questions
			label end
			`	"No, thank you for caring," she says. "Whatever you do, just be cautious about it. People are scared, and I have a hunch that when they're under pressure, the Hai are a little stranger than people think they are.`
			`	"Safe travels, Captain."`
				decline
	on decline
		clear "Hai Reveal Concern Period: Raid on Allhome (local): label help"
		clear "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"

> mission "Hai Reveal Concern Period: Raid on Allhome (away)"

Reviewer 4: ```suggestion

mission "Hai Reveal Concern Period: Raid on Allhome (away)"
```

			label greet
			`	You introduce yourself. "I am Kelamursh, but you should refer to me as Stationkeeper," she says in response. "Now, if you don't mind, I believe your arrival has solved our last remaining problems, putting us in good stead to depart momentarily. If I may see your ship's accommodation we'll make final preparations and depart in an hour or so."`
			choice
				`	"Let's go."`
					goto depart
				`	"We were getting paid for this, right?"`
			`	She blinks in surprise. "Of course, didn't they tell you? Nevermind, you can confirm the details with Heyru after I've checked your vessel."`
			label depart
			`	You show her to her bunks. She seems satisfied, and immediately begins calling people to get things sorted. She does not seem to be the kind for casual conversation.`
				accept
	on accept
		set "hr: sandsnap project"
		log "People" "Stationkeeper Kelamursh" `Kelamursh is a high-ranking employee in an arms manufacturing company. With the concerns on Allhome, she was entrusted with the responsibility of reactivating Sandsnap Station and taking on the role of Stationkeeper.`

> mission "HRCP: Sandsnap Mercenaries"

Reviewer 4: ```suggestion

mission "HRCP: Sandsnap Mercenaries"
```

	landing
	invisible
	source "Makerplace"
	to offer
		has "Hai Reveal Concern Period: Reactivating Sandsnap: accepted"
	to complete
		never
	npc
		government "Hai Merchant (Human)"
		personality escort coward opportunistic vindictive
		fleet
			names "hai (human)"
			variant
				"Grasshopper (Ion)"
				"Grasshopper (Tracker)"
				"Headhunter (Hai)" 2
				"Firebird (Hai Shields)"
				"Firebird (Hai Weapons)"
				"Lightning Bug (Pulse)"
				"Corvette (Hai)"
				"Leviathan (Hai Weapons)"

> mission "Hai Reveal Concern Period: Securing Sandsnap"

Reviewer 4: ```suggestion

mission "Hai Reveal Concern Period: Securing Sandsnap"
```

		dialog "The supply ships have not all arrived safely yet."
	on fail
		dialog "The supply convoy has been lost. The Stationkeeper will be disappointed."
	npc accompany
		government "Hai Merchant"
		personality escort timid
		system "Ula Mon"
		fleet "Large Hai Merchant" 3
	npc
		government "Pirate"
		personality coward opportunistic vindictive
		system "Waypoint"
		fleet "Marauder fleet IX" 2
	on complete
		"reputation: Hai" += 200
		"reputation: Hai (Wormhole Access)" += 200
		"reputation: Hai Merchant" += 200
		"reputation: Hai Merchant (Human)" += 200
		"reputation: Hai Merchant (Sympathizers)" += 200
		event "hr: sandsnap awoken" 5
		fail "HRCP: Sandsnap Mercenaries"
		# Ideally eventually use: payment ( "armament deterrence" * 18000 - "cargo attractiveness" * 3000 )
		payment 1200000
		conversation
			`As you're re-docking behind the supply convoy, a message pings on your console. It contains a short text from "Heyru" thanking you for your assistance and advising that you've been sent a payment of <payment> for your efforts.`
			`	It seems that the security mechanisms in the docking section have been reactivated, so they no longer need their mercenaries to hang around.`

> 

Reviewer 4: ```suggestion
```
One line break before events

	planet Sandsnap
		government "Hai Merchant"
		attributes hai station
		landscape land/station35
		music ambient/machinery
		description `This station was recently re-powered by a private military contractor to serve as a resupply and repair base that Hai warships can be stationed at to respond more rapidly to pirates entering the system. There has been a constant presence here since it was set up shortly after the pirate raids on Allhome.`
		spaceport `The operating model for the spaceport is evidently based on the military base at Everhope, but Sandsnap's layout makes restricting portions of the promenade unfeasible. While this leaves the dual-level promenade fully open to civilian access, security restrictions have been put in place at all access points to the higher levels.`
		`	The floor has had new texturing laid down to prevent slipping, but nearly every other surface appears to be coated in a fine, sticky, metallic powder. Frequently touched surfaces have been polished clean, but otherwise any accidental contact with the powder results in fingerprints being left on any non-metallic surface until it is thoroughly wiped off.`
		bribe 0.8
		security 0.8

> mission "Hai Reveal Concern Period: Sandsnap Failed"

Reviewer 4: ```suggestion

mission "Hai Reveal Concern Period: Sandsnap Failed"
```

		add fleet "Large Core Pirates" 12000
		add fleet "Anti-Pirate Hai (Wormhole Travel)" 2000

event "hr: concern period"

mission "Hai Leaks Response 1"
	landing
	invisible
	source
		government "Hai"
		attributes "spaceport"
	to offer
		has "event: hr: pirates respond to leaks"
	on offer
		event "hr: pirate uptick" 1
		conversation
			`As you go about your business you catch sight, briefly, of a chart in a news segment reporting that Hai Security forces have recorded a day-on-day increase of eleven percent in the number of human pilots being detained for smuggling contraband.`
			`	The only other detail you are able to glean is that this appears evenly spread between known pilots and newcomers. The spoken part is all in Hai though, leaving you with nothing more than the dual-language annotations to go off.`
				decline

event "hr: pirate uptick"

mission "Hai Reveal Concern Period: Humans Going Home"
	landing
	passengers 5 10
	source
		government "Hai"
		not attributes "uninhabited" "retirement"
	destination
		attributes "hai: human presence"
	to offer
		has "event: hr: pirate uptick"
		random < 10
		"passenger space" > 9
	on offer
		conversation

> 			`As you land you are hailed separately by one of the Hai in the local control center. "Captain, apologies for calling you on a non-control related matter, but I noticed that you've got capacity to potentially help resolve a situation that's developing."`

Reviewer: ```suggestion
			`As you land you are hailed separately by one of the Hai in the local control center. "Captain, apologies for calling you on a non-control related matter, but I noticed that you have the capacity to help resolve a situation that's developing."`
```

Author: ```suggestion
`As you land you are hailed separately by one of the Hai in the local control center. "Captain, apologies for calling you on a non-control related matter, but I noticed that you may have the capacity to help resolve a situation that's developing."`
```

		spaceport `The operating model for the spaceport is evidently based on the military base at Everhope, but Sandsnap's layout makes restricting portions of the promenade unfeasible. While this leaves the dual-level promenade fully open to civilian access, security restrictions have been put in place at all access points to the higher levels.`
		`	The floor has had new texturing laid down to prevent slipping, but nearly every other surface appears to be coated in a fine, sticky, metallic powder. Frequently touched surfaces have been polished clean, but otherwise any accidental contact with the powder results in fingerprints being left on any non-metallic surface until it is thoroughly wiped off.`
		bribe 0.8
		security 0.8

mission "Hai Reveal Concern Period: Sandsnap Failed"
	landing
	invisible
	source "Sandsnap"
	to offer
		has "Hai Reveal Concern Period: Securing Sandsnap: Failed"
	on offer
		log `Failed to escort the convoy safely to Sandsnap. The station will not be reactivated any time soon.`
		conversation

> 			`The Stationkeeper is distraught and palms you off to her subordinate who confirms that there is no payment he can give you.`

Reviewer: ```suggestion
			`The Stationkeeper is distraught and palms you off to her subordinate, who confirms that there is no payment he can give you.`
```
Seems to be better?

> 			`	He explains that without the initial convoy arriving safely, there are now loans that were available contingent on success and that they simply don't have the money. It will take some time to raise money for a second attempt, if there is one.`

Reviewer: ```suggestion
			`	He explains that, without the initial convoy arriving safely, there are now loans that were available contingent on success and that they simply don't have the credits. It will take some time to raise the capital for a second attempt, if there is one.`
```

				decline

mission "Hai Leaks Response 2"
	name "Go to Hai-home"
	description "Check in with the diplomats on <destination>."
	landing
	source
		government "Hai"
		attributes "spaceport"
		not planet "Hai-home"
	destination "Hai-home"
	to offer
		has "event: hr: concern period"
		or
			has "Hai Reveal Concern Period: Raid on Allhome (away): declined"
			has "Hai Reveal Concern Period: Raid on Allhome (local): declined"
		or
			has "Hai Reveal Concern Period: Reactivating Sandsnap: declined"
			has "Hai Reveal Concern Period: Securing Sandsnap: done"
			has "Hai Reveal Concern Period: Securing Sandsnap: failed"
	on offer
		conversation

> 			`A brief announcement in the regular news feeds advises that the Hai government is close to taking next actionable steps towards resolving the current situation.`
> 			`	After a moment you realize it's now been over two months since you last saw them.`
> 			`	A moment later, almost as if the universe was responding to your line of thought, a message arrives for you from Sayari. It reads; "Please return to Hai-home if you are able."`

Reviewer: ```suggestion
			`A brief announcement in the regular news feeds advises that the Hai government and the human diplomats on Hai-home are close to taking next actionable steps towards resolving the current situation. As you listen, you realize it's now been over two months since you last saw them.`
			`	A moment later, almost as if the universe was responding to your line of thought, a message arrives for you from Sayari. It reads: "Please return to Hai-home if you are able."`
```

			`	He huffs a slight wheeze. "I tell you, this is my first day and my last doing this. Goodness knows how that lump is still faster than me."`
				goto "firmly departs"
			label luck
			`	He breathes deeply before responding. "Thanks, I'll need it. It's horribly unfair that that lump is still faster than me, and I've been pressed into helping him vet every human arrival today.`
			`	"Honestly, if you see anyone concerning, it'd do me a great help if you could report them before I have to get dragged across the spaceport again."`
			label "firmly departs"
			`	He draws himself up straighter and firmly grips the tablet. "Anyway, best get on with it."`
			`	Limping away with determination, he quickly disappears from your line of sight.`
				decline

mission "Hai Reveal Exploration Period: Local Human Merchants Speak Up"
	minor
	landing
	invisible
	source
		government "Hai"

> 		not attributes "uninhabited" "retirement"

Reviewer 5: Could not you remove this condition and the one above it, seeing as how the "hai: human presence" covers it? If not, at least do this:
```suggestion
		not attributes "retirement"
```

				decline
			
			label "second" # Second option start.
			`	The captain turns out to be a younger guy with a sleek, shiny and somewhat new-looking Clipper. He clasps your hand enthusiastically in greeting. "Hail, you're not a familiar face around here. You new in these parts?"`
			choice
				`	"Yes, I arrived recently after hearing the news."`
				`	"No, I've been familiar with Hai space for quite some time."`
					goto "not new 2"
			`	The young captain nods. "Ah yeah, you look the part for a newcomer. I imagine you'll have noticed the pirates on the way here are a little bit more keen than most, but you should be aware that it's not just attacks on those in transit that are causing concerns at the moment. There are a lot more questionable characters in Hai space these days. Folks like you who've found the place following the news, and come back for less savory reasons. It's causing people some concerns, to say the least."`
			choice
				`	"I've seen; it's been getting worse lately."`
				`	"I appreciate the heads-up."`

> 			`	The young captain's tone takes on a more pressing note as he further explains, "It's not just the usual reaction though, it's more unsettling to the Hai than you might expect. It's not just a few humans that aren't great making us look bad to some of the more conservative Hai. Their attitudes are more group-oriented than we are. Humans might notice unusual or negative things more and assume that everyone in the group acts the same way, even if it's not true. We know that's not how it actually works though. For us that's just personal bias. Recognizable groups of Hai do actually tend to all act in similar ways. So, if they're not experienced with humans, it's hard for them to see the differences."`

Reviewer 5: ```suggestion
			`	The young captain's tone takes on a more pressing note as he explains further, "It's not just the usual reaction though, it's more unsettling to the Hai than you might expect. It's not just a few humans that aren't great making us look bad to some of the more conservative Hai. Their attitudes are more group-oriented than we are. Humans might notice unusual or negative things more and assume that everyone in the group acts the same way, even if it's not true. We know that's not how it actually works though. For us that's just personal bias. Recognizable groups of Hai do actually tend to all act in similar ways. So, if they're not experienced with humans, it's hard for them to see the differences."`
```

			`	He pauses. "Maybe you've seen, maybe not, but their attitudes are more group-oriented than we are. Humans might notice unusual or negative things more and assume that everyone in the group acts the same way, but it's usually not true. Recognizable groups of Hai do actually tend to all act in similar ways. So, those who aren't experienced with humans find it hard to see the differences."`
			
			label "suspicion explained"
			choice
				`	"I'll keep an eye out for any suspicious activity."`
				`	"So it's up to us to find solutions?"`
					goto "solutions"
			`	He nods. "Good, and if you want my recommendation; either report it, or distance yourself from it. There's trouble brewing and I, for one, intend on staying on their good side."`
				goto "second end"
			label "solutions"
			`	He offers you a warm smile before saying, "Well, at a minimum, to be visibly separate from the problem. Maybe that's a partial solution all by itself."`

> 				goto "second end"

Reviewer 5: ```suggestion
```

				decline
			
			label "last go" # Third option start.
			# If the dice did not see fit to give you an encounter the first two times, the third time will trigger this.
			`While briefly reviewing the job board, a woman sidles up next you and scans it with clear purpose. "Ah ha!" she says, tapping on her pad, and a moment later a passenger transport job that didn't pay particularly well changes status to 'claimed'.`
			choice
				`	"You seem very focused?"`
				`	(Say nothing.)`
					goto "bragging"
			`	She turns to face you. "A girl gotta know both what she wants and how to make a good profit in this economy. Can't be lagging."`
			choice
				`	"Sure, good luck to you.`
					goto "bragging"
				`	"What business model makes that a good job?"`
			`	With a certain swagger she leans in and taps her nose. "Let's just say that when you transport the oddballs around here, the trip tends to be more rewarding than is advertised up front. It all depends on who you know, if you know what I mean."`
			choice
				`	"Who do you need to know?"`
					goto "need to know"
				`	"I'm not interested in anything illegal."`
				`	"Aren't Hai all a bit odd?"`
					goto "oddballs"

> 			`	"Bah," she scoffs. Illegal depends on what jurisdiction you answer to, profit is universal."`

Reviewer 5: ```suggestion
			`	"Bah," she scoffs. "Illegal depends on what jurisdiction you answer to, profit is universal."`
```

			label "finish"
			choice
				`	"It's a shame that I missed it."`
				`	"Will there be more auctions?"`
					goto "more auctions"
			`	"Not really," he shakes his head. "These contracts are fiddly. Often require knowledge of local contacts to make really good profit. Very strict timelines too. There was only one that I could bid on confidently, but my cargo was too small to compete with winner of that one. I will go now, but maybe visit Greenview some time, it very pretty."`
				decline
			label "more auctions"
			`	"Unlikely," he shakes his head. "These contracts quite fiddly, quite hard, need to know contacts beforehand to make really good profit. Merchants take them only when very confident. If some small handful fall through, agency usually select specific captain from back-up list for fulfillment. Cost agency extra, but make sure job gets done properly and on time. I will go now, but maybe visit Greenview some time, it very pretty."`
				decline
			label "depart"

> 			`	Whatever is happening isn't worth your time.`

Reviewer 5: ```suggestion
			`	Whatever is happening probably isn't worth your time.`
```

			`	Another one steps forward, "We were hoping that the small amount of humans was a good business opportunity, which was true... until recently. Indeed, my colleagues and I wish to return to <planet> where we used to do business in the past, but you understand we cannot just ask any captain to take us, even if we are willing to make do without proper accommodations due to the relative urgency of the situation."`
			choice
				`	"Of course, welcome aboard."`
				`	"Sorry, you're on your own."`
					goto "rejected"
			`	You proceed to show them all their bunks, which they describe as "satisfactory".`
				accept
			label "not now"
			`	The Hai seems disappointed. "Very well Captain, I understand. Hopefully someone else comes along."`
				defer
			label "not ever"
			`	"Very well Captain, enjoy your stay." The Hai tuts before disconnecting the call.`
				decline
			label "rejected"
			`	You leave Brin spluttering in shock and manage to draw a number of very dark looks from nearby Hai.`
				decline
	on complete
		payment 100000

> 		dialog `The little group of business partners depart from your ship, and their representative addresses you again, "Very pleasant little ride, Captain. We have transferred <payment> as agreed, and we were even able to conduct some lucrative business opportunities from within your ship. Until next time, <last>."`

Reviewer 5: ```suggestion
		dialog `The little group of business partners depart from your ship, and their representative addresses you again, "A very pleasant little ride, Captain. We have transferred <payment> as agreed, and we were even able to conduct some lucrative business opportunities from within your ship. Until next time, <last>."`
```

			`	He collects a device from a carry crate and hands it to you. "Now that we've got the basics up and running, the station is in safe mode. You'll need to plug this into your communications system to re-dock safely. The supply convoy has them pre-installed."`
			choice
				`	(Go immediately.)`
					accept
				`	(Loiter to find out more.)`
					goto loiter
				`	"What were we getting paid for this?"`
			`	He frowns. "It was based on your ship's combat quality. I don't have that tablet with me right now though. Once the supply ships have arrived I should have access to that again. Now go on, you won't get paid at all if the supply ships don't make it."`
				accept
			label loiter
			`	He blinks at you. "Well, what are you waiting for? Go! Come back when all the supply ships have arrived safely."`
			`	You are left with little choice other than to nod and get on your way.`

> 				accept

Reviewer 5: ```suggestion
				launch
```
He literally just booted you out.

	npc accompany
		government "Hai Merchant"
		personality escort timid
		system "Ula Mon"
		fleet "Large Hai Merchant" 3
	npc
		government "Pirate"
		personality coward opportunistic vindictive
		system "Waypoint"
		fleet "Marauder fleet IX" 2
	on complete
		"reputation: Hai" += 200
		"reputation: Hai (Wormhole Access)" += 200
		"reputation: Hai Merchant" += 200
		"reputation: Hai Merchant (Human)" += 200
		"reputation: Hai Merchant (Sympathizers)" += 200
		set "hr: sandsnap project completed"
		event "hr: sandsnap awoken" 5
		fail "HRCP: Sandsnap Mercenaries"
		# Ideally eventually use: payment ( "armament deterrence" * 18000 - "cargo attractiveness" * 3000 )
		payment 1200000
		conversation
			`As you're re-docking behind the supply convoy, a message pings on your console. It contains a short text from "Heyru" thanking you for your assistance and advising that you've been sent a payment of <payment> for your efforts.`
			`	It seems that the security mechanisms in the docking section have been reactivated, so they no longer need their mercenaries to hang around.`

event "hr: sandsnap awoken"
	planet Sandsnap

> 		government "Hai Merchant"

Reviewer 5: Merchant?
```suggestion
		government "Hai"
```

		attributes hai station
		landscape land/station35
		music ambient/machinery

> 		description `This station was recently re-powered by a private military contractor to serve as a resupply and repair base that Hai warships can be stationed at to respond more rapidly to pirates entering the system. There has been a constant presence here since it was set up shortly after the pirate raids on Allhome.`

Reviewer 5: First sentence again feels run-ony.

			label "she ok"
			`	"I will be fine, though it is challenging adapting to how humans deal with crisis and trauma," she says. "No doubt they will be having a far worse time of things on Allhome. Still..."`
			
			label uncle
			`	She looks wistfully towards the docks. "His uncle is meant to be returning from having gone to pay off the final sum on his debt and wrap up some final business in human space. With the current events, we are both very worried he will not return, though I am doing my best to pretend otherwise."`
			`	She settles back in her seat with deliberate dignity.`
				goto leave
				
			label peace
			`	"Thank you, but it is okay, Captain.`
			label leave
			`	"Do not trouble yourself over having questioned us. There is no shortcut for processing trauma, and it does the boy no good to hide from the world."`
			`	She gestures at the screen. "No doubt many people will have many ideas for how to help things soon enough. If it troubles you, lend them a hand. Now, be about your business and farewell."`
			label walking
			`	Barely ten minutes later there is already someone on the screen arguing for a range of measures to help with the pirate situation.`
				decline

> mission "HR: Call for Aid with Sandsnap"

Reviewer: Is this a job that the player can accept multiple of to go to Makerplace and get paid for just going there? The `repeat` probably should be removed.

I think I could add some conversation here about being introduced to the Sandsnap Project.

Author: The repeat is needed for it to show up each time I believe.
If the player is going many places then, theoretically, yes. It stops offering as soon as you get there though.

	name "Join an effort on <planet>"
	job
	repeat
	description "Experienced combat pilots are requested to apply at the spaceport on <destination> for a project to reactivate the Sandsnap station."
	to offer
		not "hr: sandsnap project"
		not "Hai Reveal Concern Period: Reactivating Sandsnap: declined"
	source
		government "Hai"
		not planet "Makerplace"
	destination "Makerplace"
	to complete
		has "hr: sandsnap project"
	on complete
		payment 1000 150

mission "Hai Reveal Concern Period: Reactivating Sandsnap"
	priority

> 	name "Transport the Stationkeeper to <planet>."

Author: Does it add the full stop automatically with <planet>?

> 	name "Transport the Stationkeeper to <planet>."

Reviewer: I don't think mission names tend to have punctuation, except ones like `What happened to Rand?`

			label depart
			`	You show her to her bunks. She seems satisfied, and immediately begins calling people to get things sorted. She does not seem to be the kind for casual conversation.`
				accept
	on accept
		set "hr: sandsnap project"
		log "People" "Stationkeeper Kelamursh" `Kelamursh is a high-ranking employee in an arms manufacturing company. After the raids on Allhome she was entrusted with the responsibility of reactivating Sandsnap Station and taking on the role of Stationkeeper.`

mission "HRCP: Sandsnap Mercenaries"
	landing
	invisible
	source "Makerplace"
	to offer
		has "Hai Reveal Concern Period: Reactivating Sandsnap: accepted"
	to complete
		never
	npc
		government "Hai Merchant (Human)"
		personality escort coward opportunistic vindictive
		fleet
			names "hai (human)"
			variant
				"Grasshopper (Ion)"
				"Grasshopper (Tracker)"
				"Headhunter (Hai)" 2
				"Firebird (Hai Shields)"
				"Firebird (Hai Weapons)"
				"Lightning Bug (Pulse)"
				"Corvette (Hai)"
				"Leviathan (Hai Weapons)"

mission "Hai Reveal Concern Period: Securing Sandsnap"

> 	name "Defend convoy arriving in <system>."

Reviewer: ```suggestion
	name "Defend convoy arriving in <system>"
```

			`	He collects a device from a carry crate and hands it to you. "Now that we've got the basics up and running, the station is in safe mode. You'll need to plug this into your communications system to re-dock safely. The supply convoy has them pre-installed."`
			choice
				`	(Go immediately.)`
					accept
				`	(Loiter to find out more.)`
					goto loiter
				`	"What were we getting paid for this?"`
			`	He frowns. "It was based on your ship's combat quality. I don't have that tablet with me right now though. Once the supply ships have arrived I should have access to that again. Now go on, you won't get paid at all if the supply ships don't make it."`
				accept
			label loiter
			`	He blinks at you. "Well, what are you waiting for? Go! Come back when all the supply ships have arrived safely."`
			`	You are left with little choice other than to nod and get on your way.`
				accept
	on accept
		set "hr: sandsnap project"
	on visit

> 		dialog "The supply ships have not all arrived safely yet."

Reviewer: ```suggestion
		dialog "The supply ships have not all arrived safely yet. Depart and ensure they are in the <system> system."
```
Hmm, I'm not entirely sure if this will display the right thing since the mission has no destination.

I could get an `<origin system>` substitution, though...

	on fail

> 		dialog "The supply convoy has been lost."

Reviewer: Should there be some sort of second chance or backup scenario in this case?

Author: Nope, but a conversation follow-up might be worthwhile.

Author: Left down.

			`	"Ah, Captain, sorry about all that. What can I do for you?"`
			choice
				`	"Did you have to fight that man?"`
				`	"What's with the sign?"`
					goto "shop sign"
			`	He takes a deep breath to steady himself before responding. "Unfortunately, yes. Fortunately for me, he was one of those captains who forget that walking up and down the length of your ship doesn't really qualify as regular exercise.`
			label "shop sign"
			`	"It's all a very unfortunate situation at the moment. With the increased appearance of humans in our space, and the uncertainty over what's going on, some merchants aren't plying the trade they usually do back and forth. The supplies of a few select things that are imported from the Republic have not been appearing quite as often, and so some Hai have taken to stockpiling certain commodities in case the supply shuts off entirely." He looks pointedly at the snus packs. "Tobacco is one of those commodities. Regulations prevent it from being grown on any Hai worlds. So, unless someone goes to the effort of setting up a greenhouse in space, that's entirely imported."`

> 			`	He shrugs. "Personally, I think it's a gross habit, but it's far from being the only addictive substance around. It gives Hai a slightly improved intuition for maths by the way, not quite the same effect as on humans. Which is why it's popular with merchants, but that captain has had some bad luck, and they'd been sold out at his last three stops."`

Reviewer 3: "Math" is indeed the US version of "maths."

Adding "but" really isn't appropriate with how it changes the meaning here, though, because "not quite the same effect as humans" is an explanation of the math boosting, not a contradiction to it.

> 			`	He shrugs. "Personally, I think it's a gross habit, but it's far from being the only addictive substance around. It gives Hai a slightly improved intuition for maths by the way, not quite the same effect as on humans. Which is why it's popular with merchants, but that captain has had some bad luck, and they'd been sold out at his last three stops."`

Reviewer 6: Ah, I see. How about just the maths change then?
```suggestion
			`	He shrugs. "Personally, I think it's a gross habit, but it's far from being the only addictive substance around. It gives Hai a slightly improved intuition for math by the way, not quite the same effect as on humans. Which is why it's popular with merchants, but that captain has had some bad luck, and they'd been sold out at his last three stops."`
```

				goto "firmly departs"
			label luck
			`	He breathes deeply before responding. "Thanks, I'll need it. It's horribly unfair that that lump is still faster than me, and I've been pressed into helping him vet every human arrival today.`
			`	"Honestly, if you see anyone concerning, it'd do me a great help if you could report them before I have to get dragged across the spaceport again."`
			label "firmly departs"
			`	He draws himself up straighter and firmly grips the tablet. "Anyway, best get on with it."`
			`	Limping away with determination, he quickly disappears from your line of sight.`
				decline

mission "Hai Reveal Exploration Period: Local Human Merchants Speak Up"
	minor
	landing
	invisible
	source
		government "Hai"
		not attributes "uninhabited" "retirement"
		attributes "hai: human presence"
		not planet "Hai-home"
	to offer
		has "Hai Reveal: Meet the Team: Sayari 2: done"
		not "event: hr: exploration period"
		random < 30
		"HREP: Local Human Merchants (defer count)" < 2
	on defer
		"HREP: Local Human Merchants (defer count)" ++
	on offer
		conversation
			branch "last go"
				"HREP: Local Human Merchants (defer count)" = 2

> 			`While organizing your ship to be refueled, a distinctly not-Hai conversation that the captain of the ship parked in the next bay is having with one of his crew can be heard.`

Author: ```suggestion
			`While organizing for your ship to be refueled, a distinctly not-Hai conversation that the captain of the ship parked in the next bay is having with one of his crew can be heard.`
```

			`	They look like they're almost ready to depart.`
			choice
				`	(Approach them.)`
				`	(Call out to them.)`
				`	(Leave them be.)`
					defer
			branch "roll the dice"
				random < 50
			`	They notice you but just wave you off and board their ship. They're clearly not in the mood for chatting this time.`
				defer
			
			label "roll the dice"
			branch "second"
				"HREP: Local Human Merchants (defer count)" = 2
			# First option on first encounter if the dice decide it is so, second encounter goes to second option.

> 			`	The captain, who turns out to be an older guy, greets you, "Good day, Captain. You new in these parts?"`

Reviewer: It's more that a greeting took place?

The comma might work if the dialog was written like this:
```suggestion
			`	The captain, who turns out to be an older guy, says to you, "Good day, Captain. You new in these parts?"`
```

			choice
				`	"You know I can hear you, right?"`
				`	"But Hai space isn't even that profitable?"`
					goto "not profitable"
			`	"'Course I do, just don't care. This place ain't that profitable to start with. I'm here cause Greenwater's a nice place to live for three months of the year. Now all these new folks like yourself are just making it harder to earn a living and increasing the number of hazards I gotta pay to deal with."`
				goto "stinkeye"
			label "not profitable"
			`	"'Course not, that's my whole point, genius. I'm here cause Greenwater's a nice place to live for three months of the year. Now all these new folks like yourself are just making it harder to earn a living and increasing the number of hazards I gotta pay to deal with."`
			label "stinkeye"
			`	His crewman cuts in, "Don't mind him, he's always like this."`
			`	The captain gives you both the stinkeye and harrumphs. "Get back on board, we're late."`
			`	As he stomps off he points at you. "Just remember, your lot are making our lives harder. Keep it up and something's gonna go wrong." He seals the airlock behind himself before giving you a chance to respond.`
				decline
			
			label "not new 1"

> 			`	"Ahah, you'll know what I'm talking about then, Captain." He glances back at his crewman. "All these new folks in their clunkers from Lionheart and whatnot. Filling up the space lanes and splitting the already meager profits."`

Author: ```suggestion
			`	"Ah-ha, you'll know what I'm talking about then, Captain." He glances back at his crewman. "All these new folks in their clunkers from Lionheart and whatnot. Filling up the space lanes and splitting the already meager profits."`
```

> 			`	"Ahah, you'll know what I'm talking about then, Captain." He glances back at his crewman. "All these new folks in their clunkers from Lionheart and whatnot. Filling up the space lanes and splitting the already meager profits."`

Reviewer 7: 👍 Ah-ha works.

> 			`	"Ahah, you'll know what I'm talking about then, Captain." He glances back at his crewman. "All these new folks in their clunkers from Lionheart and whatnot. Filling up the space lanes and splitting the already meager profits."`

Reviewer 6: Yeah that's good.

			choice
				`	"I've seen; it's been getting worse lately."`
				`	"I appreciate the heads-up."`
			`	The young captain's tone takes on a more pressing note as he further explains, "It's not just the usual reaction though, it's more unsettling to the Hai than you might expect. It's not just a few humans that aren't great making us look bad to some of the more conservative Hai. Their attitudes are more group-oriented than we are. Humans might notice unusual or negative things more and assume that everyone in the group acts the same way, even if it's not true. We know that's not how it actually works though. For us that's just personal bias. Recognizable groups of Hai do actually tend to all act in similar ways. So, if they're not experienced with humans, it's hard for them to see the differences."`
				goto "suspicion explained"
			
			label "not new 2"
			`	"Ah, have you noticed it then?" He glances out at the broader docking area. "The suspicion? The uncertainty?"`
			choice
				`	"I suppose..."`
				`	"Not really, no."`
					goto "not seen"
			`	"If you're not sure, you probably haven't really seen it yet."`
			label "not seen"

> 			`	The young captain sighs and shakes his head. "I suppose it's not obvious yet, maybe it's because I mostly transport passengers. Still, there are a lot more questionable characters in Hai space these days. Folks who've found the place following the news, and come back for less savory reasons. It's causing people some concerns, to say the least."`

Reviewer 5: This seems reasonable.

			`	He pauses. "Maybe you've seen, maybe not, but their attitudes are more group-oriented than we are. Humans might notice unusual or negative things more and assume that everyone in the group acts the same way, but it's usually not true. Recognizable groups of Hai do actually tend to all act in similar ways. So, those who aren't experienced with humans find it hard to see the differences."`
			
			label "suspicion explained"
			choice
				`	"I'll keep an eye out for any suspicious activity."`
				`	"So it's up to us to find solutions?"`
					goto "solutions"
			`	He nods. "Good, and if you want my recommendation; either report it, or distance yourself from it. There's trouble brewing and I, for one, intend on staying on their good side."`
				goto "second end"
			label "solutions"
			`	He offers you a warm smile before saying, "Well, at a minimum, to be visibly separate from the problem. Maybe that's a partial solution all by itself."`
				goto "second end"
			label "second end"

> 			`	His crewperson finishes securing a crate and calls out, "Come on Captain, we're running late."`

Reviewer 5: So does this.

			choice
				`	"Sure, good luck to you.`
					goto "bragging"
				`	"What business model makes that a good job?"`
			`	With a certain swagger she leans in and taps her nose. "Let's just say that when you transport the oddballs around here, the trip tends to be more rewarding than is advertised up front. It all depends on who you know, if you know what I mean."`
			choice
				`	"Who do you need to know?"`
					goto "need to know"
				`	"I'm not interested in anything illegal."`
				`	"Aren't Hai all a bit odd?"`
					goto "oddballs"
			`	"Bah," she scoffs. Illegal depends on what jurisdiction you answer to, profit is universal."`
				goto "third end"
			label "need to know"
			`	"Let's just say that these groups tend to be pretty happy no matter where you take them along the way." She chuckles. "There are... opportunities... for unusual visitors in the right places."`
				goto "third end"
			label "oddballs"
			`	"Nah, nah, the really odd ones. The cosplay spies and the danger tourists. That sort of thing." She rolls her eyes. "That lot are happy no matter where you take them along the way."`
				goto "third end"
			
			label "bragging"

> 			`	She winks at you. "More profit in those than you might think ya know, but hey, more for me."`

Reviewer 6: This sentence feels weirdly run-on without the comma.

> 			`	She winks at you. "More profit in those than you might think ya know, but hey, more for me."`

Reviewer 3: The comma should be there, grammatically, but as dialogue, it's more about the pacing the character says it with.

> 			`	She winks at you. "More profit in those than you might think ya know, but hey, more for me."`

Reviewer 6: Yeah fair.

			choice
				`	"What are you on about?"`
					goto "on about"
				`	(Ignore her.)`
			`	She leans in uncomfortably close to you. "You can take them anywhere," she breathes, barely audible but with distinct emphasis on 'anywhere'.`
				goto "slip"
			label "on about"

> 			`	"Ahah, wouldn't you like to know? But you'll have to take one to find out!"`

Author: ```suggestion
			`	"Aha, wouldn't you like to know? But you'll have to take one to find out!"`
```

		random < 10
	on offer
		conversation
			`The spaceport feels a little quiet as you enter, but it seems that everyone seems to be coming or going from the same direction.`
			choice
				`	(Check it out.)`
				`	(Ignore it.)`
					decline
			`	Following along it becomes clear that there is a large gathering adjacent to the job services section. A number of Hai are clamoring for attention while several others appear to be in intense discussion. Notes are changing hands and numbers are being thrown up and you realize suddenly that, while the format is a little odd, this is an auction of some sort.`
			`	A moment later there's a cry that rings out in Hai, followed by a second voice calling, "A done deal." As you watch, a human wearing very non-human clothing steps up, takes something, and proceeds into a room behind the podium this was being led from. There are a number of grumbles and the crowd immediately begins to disperse.`
			choice
				`	(Ask someone about the auction.)`
				`	(Leave.)`
					goto "depart"
			`	You manage to flag down a younger Hai who doesn't appear to be too disappointed in the outcome. He looks at you and asks with a terrible accent, "What you want?"`

> 			`	After briefly explaining that you want to know what the auction was about, he thinks for a second and answers. "In your worlds you have place called zuu, no, zoos. Yes, zoos, with animals and such? We have station called Greenview. Hei Low Io run special zoo there with rare species from everywhere. Current situation make them nervous about supply lines and trade with Republic, so they put out bounty for supplies, stock up in case of diplomatic catastrophe or war."`

Author: It is, however, intentionally more generalised.

The catastrophe could involve pirates, Unfettered, the Syndicate, etc. it's not localised.

			choice
				`	"They think there will be a war?"`
				`	"How do you put a bounty on supplies?"`
					goto "bounty on supplies"

> 			`	He shakes his head, "No, not think war, they prepare for anything. They are vulnerable organization. Small disruption of supply for important thing make them lose many millions of credits if artificial ecosystem fails, or specific food nutrient cause species death. Very, very, sensitive to possible shocks they are. Always first to take action when politics is uncertain. Always first to prepare. Usually right to prepare.`

Author: If you look at what's said about the Hai language, it's specifically particles and joining words that are the "difficult" part of Hai speaking Human.

	on offer
		conversation
			`As you land you are hailed separately by one of the Hai in the local control center. "Captain, apologies for calling you on a non-control related matter, but I noticed that you've got capacity to potentially help resolve a situation that's developing."`
			choice
				`	"What's the matter?"`
				`	"I'm sorry, I'm busy right now."`
					goto "not now"
			`	"Well, there are a number of humans trying to get transport to somewhere else, but they're unwilling to book passage with any Hai captains. They've set up camp in the spaceport and they've been there long enough that it's causing some tension. We don't want to take direct action, but I think these <passengers> humans would be much better off if you could talk to them, and maybe at least take them to <destination>."`
			choice
				`	"I'll talk to them."`
				`	"Sorry, I'm not going that way."`
					goto "not ever"
			`	As it turns out, it is almost impossible to miss these humans, as their camp is located directly in the crossroads of the spaceport's major thoroughfares. Judging by their expensive clothing and somewhat impatient attitude they are people of means, which makes this even more odd.`

> 			`	As soon as it becomes apparent that you're approaching them, one comes forward to greet you. "Ah, I trust you are here because you've been encouraged to help us. I am Brin." You introduce yourself in response, and they continue, "We have run an import services business here for some time, but with the recent events suspicion of human captains has arisen. Some Hai seem to believe any human captain is potentially someone who will abduct them." They say with a chuckle. "As you can imagine, we've been here a long time, so when we were asked to vet a few arrivals first we were happy enough to do so. It's gotten out of hand, though. The other day we even had the Portmaster insisting that we vet every human captain to arrive, but we don't have time for that, and we don't have access to any services that would really let us do that reliably. So we had to put our foot down and refuse.`

Author: It needs to be a break because it was already a continuation.

						not "First Contact: Unfettered: offered"
						not "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
				`	"Thank you for your time."`
					goto end
					to display
						or
							has "Hai Reveal Concern Period: Raid on Allhome (local): label help"
							has "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
			label unfettered
			action
				set "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
			`	She seems surprised that you asked and responds, "Oh, they're a sort of rebel Hai group. If you don't know you could go visit Cloudfire in the Wah Ki system up north, or maybe even Darkcloak if you dare. A short visit would explain it far better than I ever could.`
			`	"Be very careful if you do though, they're very likely to shoot at you."`
				goto questions
			label help
			action
				set "Hai Reveal Concern Period: Raid on Allhome (local): label help"
			`	She shrugs. "I've heard talk of some things that have been suggested to help, but I don't know myself. I guess you should just do the right thing, whatever that is."`
			`	She seems to be at a loss for what else to say.`
				goto questions
			label end

> 			`	"No, thank you for caring," she says. "Whatever you do, just be cautious about it. People are scared, and I have a hunch that when they're under pressure, the Hai are a little stranger than people think they are.`

Reviewer: "Stranger" seems weird to say about an alien species; shouldn't it be "stronger"?

Author: No, because the Hai are the aliens who explicitly seem not all that strange. Especially the ones who are used to us.
This is a reminder that even though they're more or less "the same" to deal with on the surface, that they're actually *not* the same underneath - a detail that's easy to forget when you're used to them.

		not planet "Allhome"
		attributes "hai: human presence"
		not attributes "uninhabited" "retirement"
	to offer
		has "event: hr: pirates respond to leaks"
		has "First Contact: Unfettered: offered"
		not "Hai Reveal Concern Period: Raid on Allhome (local): offered"
		random < 20
	on offer
		set "hr: allhome raided"
		conversation
			`A wall screen situated between two food stands just outside the docking area is playing through a news segment, and your ears suddenly perk up at the word "marauder".`
			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew hijacked a vessel with some of the crew still on board and made off with it successfully. At this time it is unclear what has happened to those crewhai."`
			`	At a table nearby, a Hai says, "Oh, that's awful!" and the human next to her responds, "Feels like being back on Solace."`
			choice
				`	(Approach the pair.)`
				`	(Keep walking.)`
					goto walking
			

> 			`	As you approach the human, a relatively young-looking man, glances up at you. "Oh hello, just arrived did you? You didn't happen to see an Argosy out there on your way in?"`

Author: the comma throws me

			`	She settles back in her seat with deliberate dignity.`
				goto leave
				
			label peace
			`	"Thank you, but it is okay, Captain.`
			label leave
			`	"Do not trouble yourself over having questioned us. There is no shortcut for processing trauma, and it does the boy no good to hide from the world."`
			`	She gestures at the screen. "No doubt many people will have many ideas for how to help things soon enough. If it troubles you, lend them a hand. Now, be about your business and farewell."`
			label walking
			`	Barely ten minutes later there is already someone on the screen arguing for a range of measures to help with the pirate situation.`
				decline

mission "HR: Call for Aid with Sandsnap"
	name "Join an effort on <planet>"
	job
	repeat
	description "Experienced combat pilots are requested to apply at the spaceport on <destination> for a project to reactivate the Sandsnap station."
	to offer
		not "hr: sandsnap project"
		not "Hai Reveal Concern Period: Reactivating Sandsnap: declined"
	source
		government "Hai"
		not planet "Makerplace"
	destination "Makerplace"
	to complete
		has "hr: sandsnap project"

> 	on complete

Reviewer: Once you have the Stationkeeper on board it would be helpful to have a dialog telling you that the payment from this job has been transferred to you.

		payment 1000 150

mission "Hai Reveal Concern Period: Reactivating Sandsnap"
	priority
	name "Transport the Stationkeeper to <planet>"
	description "Safely bring the Stationkeeper to <destination> to reactivate the station."
	source "Makerplace"
	destination "Sandsnap"
	passengers 1
	to offer
		has "HR: Call for Aid with Sandsnap: accepted"
	blocked `You are advised that participation in the Sandsnap Project will require at least 1 bunk free for passengers.`
	on offer
		conversation
			`	Entering the spaceport requires one to walk around a very large holosign giving directions to the Sandsnap Project's office.`
			choice
				`	(Check out the project.)`
				`	(Come back later.)`
					defer
				`	(Ignore it, you've changed your mind.)`
					decline
			`	There is a greeter, or perhaps a security guard, at the door who appraises you as you approach. "You must be here for the project," he says. "Identification please."`

> 			`	After a quick check of your ID he gives you a very professional Hai smile. "Welcome to the Sandsnap Project. Your credentials are sufficient to apply so please, do come in. We hope you'll find the project interesting and the effort rewarding."`

Reviewer: I think leaving the comma out adds to the snappiness of his smile after he checks your ID.

> 			`	After a quick check of your ID he gives you a very professional Hai smile. "Welcome to the Sandsnap Project. Your credentials are sufficient to apply so please, do come in. We hope you'll find the project interesting and the effort rewarding."`

Reviewer 3: I'm reading it the same way, yea, but I think with a negative implication to that snappiness, as if the check is rushed through the same way as the sentence.

> 			`	After a quick check of your ID he gives you a very professional Hai smile. "Welcome to the Sandsnap Project. Your credentials are sufficient to apply so please, do come in. We hope you'll find the project interesting and the effort rewarding."`

Author: The check is rushed through, he's basically checking `not <one specific detail>`  - he isn't spending much time on it.

> 			`	As you enter a harried Hai gestures you over with enthusiasm. "Greetings," he checks his tablet. "Captain <last>! Your presence in these dire times is greatly appreciated, it has been at least a hundred years since we have had such a threat to our way of life! You must have noticed the spike in pirate activity. Perhaps our northern brethren have had a point all these years."`

Author: Since when did we put dashes in speech?

> 			`	As you enter a harried Hai gestures you over with enthusiasm. "Greetings," he checks his tablet. "Captain <last>! Your presence in these dire times is greatly appreciated, it has been at least a hundred years since we have had such a threat to our way of life! You must have noticed the spike in pirate activity. Perhaps our northern brethren have had a point all these years."`

Reviewer 6: Dashes can be used to indicate a pause in speech and break up a sentence into two different clauses without causing a run-on sentence.

					goto border
				`	"What are you talking about?"`
				`	"Not at all, I love politics."`
			`	He seems surprised, "Oh. Well in that case, the Unfettered have been militarizing a lot in the last 20 years, forcing us to do the same in order to protect merchants from their raids. They may be right that we have grown weak, if the success of those pirate raids is any indication; security forces were simply not ready for them. We here are in the business of ensuring this will not happen again. Mark my words, nothing like an arms manufacturer setting up a support base to really wake a few people up.`
			label border
			`	"The raids have proven that we are not prepared to ensure true security on our borders. In order to address this we want to reactivate the mothballed station of <destination>, for which we have acquired the relevant paperwork and access codes. We have already hired a few mercenaries to help with the defense of our station, mostly local humans with experienced pasts. Once operational, the station should be a major strategic advantage for us, allowing resupply and repairs of ships at a location suitable for facilitating rapid response to detected threats."`

> 			`	He indicates a door to another room, "The Stationkeeper will be out shortly. We're almost complete our final preparations, but we've been held up trying to make do with the firepower that's signed up so far. Judging by your credentials, I suspect the Stationkeeper will want to travel with you. You will need to escort them to the station and dock so they can confirm access codes function and there are no critical errors in the initial restoration process, as well as assess if any additional supplies are required."`

Author: Yours, with one additional letter added.
```suggestion
			`	He indicates a door to another room, "The Stationkeeper will be out shortly. We've almost completed our final preparations, but we've been held up trying to make do with the firepower that's signed up so far. Judging by your credentials, I suspect that Kelamursh will want to travel with you. You will need to escort them to the station and dock, so they can confirm the access codes function and that there are no critical errors in the initial restoration process, as well as assess if any additional supplies are required."`
```

> 			`	He indicates a door to another room, "The Stationkeeper will be out shortly. We're almost complete our final preparations, but we've been held up trying to make do with the firepower that's signed up so far. Judging by your credentials, I suspect the Stationkeeper will want to travel with you. You will need to escort them to the station and dock so they can confirm access codes function and there are no critical errors in the initial restoration process, as well as assess if any additional supplies are required."`

Reviewer 3: Whoops, good catch.

			`	As soon as he finishes, the door opens and a number of people file out, including a well-dressed Hai who immediately stops next to you.`
			choice
				`	(Greet the Stationkeeper.)`
					goto greet
				`	(Bow to the Stationkeeper.)`
			`	She appears flattered in a Hai sort-of way. "There is no need, Captain, my station is not operational, and this is a business operation, not an appointed position. For now, anyway."`
			label greet
			`	You introduce yourself. "I am Kelamursh, but you should refer to me as Stationkeeper," she says in response. "Now, if you don't mind, I believe your arrival has solved our last remaining problems, putting us in good stead to depart momentarily. If I may see your ship's accommodation we'll make final preparations and depart in an hour or so."`
			choice
				`	"Let's go."`
					goto depart

> 				`	"We were getting paid for this right?"`

Reviewer 6: ```suggestion
				`	"We were getting paid for this, right?"`
```

	invisible
	source "Makerplace"
	to offer
		has "Hai Reveal Concern Period: Reactivating Sandsnap: accepted"
	to complete
		never
	npc
		government "Hai Merchant (Human)"
		personality escort coward opportunistic vindictive
		fleet
			names "hai (human)"
			variant
				"Grasshopper (Ion)"
				"Grasshopper (Tracker)"
				"Headhunter (Hai)" 2
				"Firebird (Hai Shields)"
				"Firebird (Hai Weapons)"
				"Lightning Bug (Pulse)"
				"Corvette (Hai)"
				"Leviathan (Hai Weapons)"

mission "Hai Reveal Concern Period: Securing Sandsnap"
	name "Defend convoy arriving in <system>"
	description "Keep <system> secure against pirates for the supply convoy so that they can reach <planet> safely."
	landing
	source "Sandsnap"
	to offer
		has "hr: sandsnap project"
	on offer
		conversation
			`Upon arrival the Stationkeeper requests that you stay aboard your ship in case hostiles approach the station, while she suits up and goes aboard to run the access codes and start things up. You can see a selection of other Hai with various tools and equipment disembarking a few other ships in the process. Before long, the main lighting comes on in the docking section and the team disappears inside.`

> 			`	After a short while you receive a message requesting that you report to the operations center inside the main promenade. You proceed to equip your own suit and exit your ship. The station itself is well preserved, except for the lack of atmosphere. Nothing seems to have vented it, no big hole in the ceiling, and the expansive ceiling windows of the main promenade provide a crystal clear view of the station's many decks towering above you. Exterior navigation lights are gradually flicking on up the height of the station, so that walking through feels like watching a ghost city slowly come to glittering life. Emergency floor lighting has been activated, showing you the way.`

Author: The lack of a hole is authoritative, not perceived.

> 			`	After a short while you receive a message requesting that you report to the operations center inside the main promenade. You proceed to equip your own suit and exit your ship. The station itself is well preserved, except for the lack of atmosphere. Nothing seems to have vented it, no big hole in the ceiling, and the expansive ceiling windows of the main promenade provide a crystal clear view of the station's many decks towering above you. Exterior navigation lights are gradually flicking on up the height of the station, so that walking through feels like watching a ghost city slowly come to glittering life. Emergency floor lighting has been activated, showing you the way.`

Reviewer 3: That feels a bit like knowledge we may not have in this context, but either way the transition still seems better served with contesting the window view to a potential whole than the window view being an addition to the lack of hole.

Revised suggestion for just that.

> 			`	After a short while you receive a message requesting that you report to the operations center inside the main promenade. You proceed to equip your own suit and exit your ship. The station itself is well preserved, except for the lack of atmosphere. Nothing seems to have vented it, no big hole in the ceiling, but the expansive ceiling windows of the main promenade still provide a crystal clear view of the station's many decks towering above you. Exterior navigation lights are gradually flicking on up the height of the station, so that walking through feels like watching a ghost city slowly come to glittering life. Emergency floor lighting has been activated, showing you the way.`

Reviewer 6: ```suggestion
			`	After a short while you receive a message requesting that you report to the operations center inside the main promenade. You proceed to equip your own suit and exit your ship. The station itself is well preserved, except for the lack of atmosphere. Nothing seems to have vented it- you spy no big hole in the ceiling - but the expansive ceiling windows of the main promenade still provide a crystal clear view of the station's many decks towering above you. Exterior navigation lights are gradually flicking on up the height of the station, so that walking through feels like watching a ghost town slowly come to glittering life. Emergency floor lighting has been activated, showing you the way.`
```

			`	He chuckles as if having made a joke, as if you hadn't just walked beneath the open view of over three hundred decks above you. He seems oblivious to the strangeness of this, so perhaps it is a Hai thing.`
			`	"I am sure you have questions but they will have to wait until we're more established. We have supplies coming and they will be vulnerable to any pirates that may poke their noses in. We need you to get out there and ensure the supplies can all arrive here safely. Local security is still about, so it shouldn't be too problematic."`
			`	He collects a device from a carry crate and hands it to you. "Now that we've got the basics up and running, the station is in safe mode. You'll need to plug this into your communications system to re-dock safely. The supply convoy has them pre-installed."`
			choice
				`	(Go immediately.)`
					accept
				`	(Loiter to find out more.)`
					goto loiter

> 				`	"What were we getting paid for this?"`

Reviewer 6: ```suggestion
				`	"How much were we getting paid for this?"`
```

			`	You are left with little choice other than to nod and get on your way.`
				accept
	on accept
		set "hr: sandsnap project"
	on visit
		dialog "The supply ships have not all arrived safely yet."
	on fail
		dialog "The supply convoy has been lost."
	npc accompany
		government "Hai Merchant"
		personality escort timid
		system "Ula Mon"
		fleet "Large Hai Merchant" 3
	npc
		government "Pirate"
		personality coward opportunistic vindictive
		system "Waypoint"
		fleet "Marauder fleet IX" 2
	on complete
		"reputation: Hai" += 200
		"reputation: Hai (Wormhole Access)" += 200
		"reputation: Hai Merchant" += 200
		"reputation: Hai Merchant (Human)" += 200
		"reputation: Hai Merchant (Sympathizers)" += 200
		event "hr: sandsnap awoken" 5
		fail "HRCP: Sandsnap Mercenaries"
		# Ideally eventually use: payment ( "armament deterrence" * 18000 - "cargo attractiveness" * 3000 )
		payment 1200000
		conversation
			`As you're re-docking behind the supply convoy, a message pings on your console. It contains a short text from "Heyru" thanking you for your assistance and advising that you've been sent a payment of <payment> for your efforts.`

> 			`	It seems that the security mechanisms in the docking section have been reactivated, so they no longer need their mercenaries to hang around.`

Author: Consistent with the other sections it is in fact only 1 docking section. All access points are in the same section of the station.

> 			`	It seems that the security mechanisms in the docking section have been reactivated, so they no longer need their mercenaries to hang around.`

Reviewer 3: Agreement flip needs to go the other way, then. Revised suggestion :)

> 			`	It seems that the security mechanisms in the docking section have been reactivated, so they no longer need their mercenaries to hang around.`

Reviewer 3: Okay, I see where I was confused now.

Good as is.

event "hr: sandsnap awoken"

> 	planet Sandsnap

Reviewer 6: Wouldn't it make more sense to use one of the mothballed stations currently in the game as a military base for HR, rather than add an entirely new one?

Reviewer: Sandsnap is an existing station.

Reviewer 6: Sorry, never mind then.

Reviewer: No problem.

							has "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
			label unfettered
			action
				set "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
			`	She seems surprised that you asked and responds, "Oh, they're a sort of rebel Hai group. If you don't know you could go visit Cloudfire in the Wah Ki system up north, or maybe even Darkcloak if you dare. A short visit would explain it far better than I ever could.`
			`	"Be very careful if you do though, they're very likely to shoot at you."`
				goto questions
			label help
			action
				set "Hai Reveal Concern Period: Raid on Allhome (local): label help"
			`	She shrugs. "I've heard talk of some things that have been suggested to help, but I don't know myself. I guess you should just do the right thing, whatever that is."`
			`	She seems to be at a loss for what else to say.`
				goto questions
			label end
			`	"No, thank you for caring," she says. "Whatever you do, just be cautious about it. People are scared, and I have a hunch that when they're under pressure, the Hai are a little stranger than people think they are.`
			`	"Safe travels, Captain."`
				decline

> 

Reviewer: This will need to be done in the future for all of HR, but I don't think there's anything stopping us from doing this one right now.
```suggestion
	on decline
		clear "Hai Reveal Concern Period: Raid on Allhome (local): label help"
		clear "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
```

			choice
				`	"Can I help you?"`
					goto polite
				`	"What's this about?"`
			`	The Hai just snorts in response.`
			label polite
			`	Glancing back over his shoulder he calls out. "Hurry up Sebastian, what do you make of this one then?"`
			`	An older gentleman, who looks quite harried, hurriedly limps over to the both of you. In his hand is a pad which he briefly consults before holding it up, apparently comparing your face to an image. With a long-suffering sigh he says, "I told you this one is fine. It was the ship coming in through 5B that was new."`
			`	The Portmaster squints at the tablet himself for a moment before giving out a huff. "Well, this one was on the way. You two, go make sure no one comes through 5B until they've been examined."`
			`	Two Hai in security uniforms immediately dash off in the direction of the offending berth, and the Portmaster gives you a brief nod and a, "Captain," before toddling off in pursuit with remarkable speed.`
			`	The human gentleman seems to be taking a second to catch his breath.`
			choice
				`	"Well, uh, good luck?"`
					goto luck
				`	"What's going on here?"`

> 			`	The gentleman rolls his eyes and draws in a long, exasperated, breath. "Madness is what's going on here. Collective madness. Some Hai got burned by some shady sorts that were new through the wormhole. Suddenly everyone's got to be vetted by a human before the Hai here will deal with them, but then there were a hundred extra people trying to cram in so every place of business could have one, and there weren't enough chairs, so some bright spark decided someone would have to follow the Portmaster around and check everyone when they came in."`

Reviewer 8: ```suggestion
			`	The gentleman rolls his eyes and draws in a long, exasperated breath. "Madness is what's going on here. Collective madness. Some Hai got burned by some shady sorts that were new through the wormhole. Suddenly everyone's got to be vetted by a human before the Hai here will deal with them, but then there were a hundred extra people trying to cram in so every place of business could have one, and there weren't enough chairs, so some bright spark decided someone would have to follow the Portmaster around and check everyone when they came in."`
```

			choice
				`	"Yes, came after the news broke."`
				`	"No, I've known about the Hai for ages."`
					goto "not new 1"
			`	He glances back at his crewman. "See, this is exactly what I was talking about. All these new folks in their clunkers from Lionheart and whatnot. Filling up the space lanes and splitting the profits."`
			choice
				`	"You know I can hear you, right?"`
				`	"But Hai space isn't even that profitable?"`
					goto "not profitable"
			`	"'Course I do, just don't care. This place ain't that profitable to start with. I'm here cause Greenwater's a nice place to live for three months of the year. Now all these new folks like yourself are just making it harder to earn a living and increasing the number of hazards I gotta pay to deal with."`
				goto "stinkeye"
			label "not profitable"
			`	"'Course not, that's my whole point, genius. I'm here cause Greenwater's a nice place to live for three months of the year. Now all these new folks like yourself are just making it harder to earn a living and increasing the number of hazards I gotta pay to deal with."`
			label "stinkeye"
			`	His crewman cuts in, "Don't mind him, he's always like this."`

> 			`	The captain gives you both the stinkeye and harrumphs. "Get back on board, we're late."`

Reviewer 8: ```suggestion
			`	The captain gives you both the stink eye and harrumphs. "Get back on board, we're late."`
```

	# Add pirate spawns to Waypoint and Heia Due
	system "Waypoint"
		government "Pirate"
		add fleet "Large Northern Pirates" 5000
		add fleet "Small Northern Pirates" 2500
		add fleet "Large Core Pirates" 6000
		add fleet "Small Core Pirates" 3000
		remove fleet "Small Hai"
		remove fleet "Large Hai"
	system "Heia Due"
		add fleet "Marauder fleet IX" 15000
		add fleet "Large Northern Pirates" 10000
		add fleet "Large Core Pirates" 12000
		add fleet "Anti-Pirate Hai (Wormhole Travel)" 2000

event "hr: concern period"

mission "Hai Reveal Concern Period: Humans Going Home"
	landing
	passengers 5 10
	source
		government "Hai"
		not attributes "uninhabited" "retirement"
	destination
		attributes "hai: human presence"
	to offer
		has "event: hr: pirates respond to leaks"
		random < 10
		"passenger space" > 9
	on offer
		conversation
			`As you land you are hailed separately by one of the Hai in the local control center. "Captain, apologies for calling you on a non-control related matter, but I noticed that you've got capacity to potentially help resolve a situation that's developing."`
			choice
				`	"What's the matter?"`
				`	"I'm sorry, I'm busy right now."`
					goto "not now"

> 			`	"Well, there are a number of humans trying to get transport to somewhere else, but they're unwilling to book passage with any Hai captains. They've set up camp in the spaceport and they've been there long enough that it's causing some tension. We don't want to take direct action, but I think these <passengers> humans would be much better off if you could talk to them, and maybe at least take them to <destination>."`

Reviewer 8: ```suggestion
			`	"Well, there are a number of humans trying to get transport to somewhere else, but they're unwilling to book passage with any Hai captains. They've set up camp in the spaceport and they've been there long enough that it's causing some tension. We don't want to take direct action, but I think these <bunks> humans would be much better off if you could talk to them, and maybe at least take them to <destination>."`
```

		not planet "Allhome"
		attributes "hai: human presence"
		not attributes "uninhabited" "retirement"
	to offer
		has "event: hr: pirates respond to leaks"
		has "First Contact: Unfettered: offered"
		not "Hai Reveal Concern Period: Raid on Allhome (local): offered"
		random < 20
	on offer
		set "hr: allhome raided"
		conversation
			`A wall screen situated between two food stands just outside the docking area is playing through a news segment, and your ears suddenly perk up at the word "marauder".`
			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew hijacked a vessel with some of the crew still on board and made off with it successfully. At this time it is unclear what has happened to those crewhai."`
			`	At a table nearby, a Hai says, "Oh, that's awful!" and the human next to her responds, "Feels like being back on Solace."`
			choice
				`	(Approach the pair.)`
				`	(Keep walking.)`
					goto walking
			

> 			`	As you approach the human, a relatively young-looking man, he glances up at you. "Oh hello, just arrived did you? You didn't happen to see an Argosy out there on your way in?"`

Reviewer 8: ```suggestion
			`	As you approach the human, a relatively young-looking man, he glances up at you. "Oh hello, just arrived, did you? You didn't happen to see an Argosy out there on your way in?"`
```
I'm not sure about that sentence.

			
			label uncle
			`	She looks wistfully towards the docks. "His uncle is meant to be returning from having gone to pay off the final sum on his debt and wrap up some final business in human space. With the current events, we are both very worried he will not return, though I am doing my best to pretend otherwise."`
			`	She settles back in her seat with deliberate dignity.`
				goto leave
				
			label peace
			`	"Thank you, but it is okay, Captain.`
			label leave
			`	"Do not trouble yourself over having questioned us. There is no shortcut for processing trauma, and it does the boy no good to hide from the world."`
			`	She gestures at the screen. "No doubt many people will have many ideas for how to help things soon enough. If it troubles you, lend them a hand. Now, be about your business and farewell."`
			label walking
			`	Barely ten minutes later there is already someone on the screen arguing for a range of measures to help with the pirate situation.`
				decline

mission "HR: Call for Aid with Sandsnap"
	name "Join an effort on <planet>"
	job

> 	repeat

Reviewer: According to the wiki:
<img width="911" alt="Screenshot 2023-11-18 at 12 03 17 PM" src="https://github.com/endless-sky/endless-sky/assets/115441627/2a16af0c-0e6e-44c3-99e2-1f7212597223">
So I feel like it should be removed.
```suggestion
```

Of course, this should be tested ingame.

	description "Experienced combat pilots are requested to apply at the spaceport on <destination> for a project to reactivate the Sandsnap station."
	to offer
		not "hr: sandsnap project"
		not "Hai Reveal Concern Period: Reactivating Sandsnap: declined"
	source
		government "Hai"
		not planet "Makerplace"
	destination "Makerplace"
	to complete
		has "hr: sandsnap project"
	on complete
		payment 1000 150
		dialog `Having accepted the job, the initial retainer of <payment> appears in your account.`

mission "Hai Reveal Concern Period: Reactivating Sandsnap"
	priority
	name "Transport the Stationkeeper to <planet>"
	description "Safely bring the Stationkeeper to <destination> to reactivate the station."
	source "Makerplace"
	destination "Sandsnap"
	passengers 1
	to offer

> 		has "HR: Call for Aid with Sandsnap: accepted"

Reviewer 4: ```suggestion
		has "HR: Call for Aid with Sandsnap: active"
```

			choice
				`	(Check out the project.)`
				`	(Come back later.)`
					defer
				`	(Ignore it, you've changed your mind.)`
					decline
			`	There is a greeter, or perhaps a security guard, at the door who appraises you as you approach. "You must be here for the project," he says. "Identification please."`
			`	After a quick check of your ID he gives you a very professional Hai smile. "Welcome to the Sandsnap Project. Your credentials are sufficient to apply so please, do come in. We hope you'll find the project interesting and the effort rewarding."`
			`	As you enter a harried Hai gestures you over with enthusiasm. "Greetings," he checks his tablet. "Captain <last>! Your presence in these dire times is greatly appreciated - it has been at least a hundred years since we have had such a threat to our way of life! You must have noticed the spike in pirate activity. Perhaps our northern brethren have had a point all these years."`
			`	He stops himself. "Not that our politics probably concern you. Let us cut to the matter at hand, shall we?"`
			choice
				`	(Nod.)`
					goto border
				`	"What are you talking about?"`
				`	"Not at all, I love politics."`

> 			`	He seems surprised, "Oh. Well in that case, the Unfettered have been militarizing a lot in the last 20 years, forcing us to do the same in order to protect merchants from their raids. They may be right that we have grown weak, if the success of those pirate raids is any indication; security forces were simply not ready for them. We here are in the business of ensuring this will not happen again. Mark my words, nothing like an arms manufacturer setting up a support base to really wake a few people up.`

Reviewer: ```suggestion
			`	He seems surprised. "Oh. Well in that case, the Unfettered have been militarizing a lot in the last 20 years, forcing us to do the same in order to protect merchants from their raids. They may be right that we have grown weak, if the success of those pirate raids is any indication; security forces were simply not ready for them. We here are in the business of ensuring this will not happen again. Mark my words, nothing like an arms manufacturer setting up a support base to really wake a few people up.`
```
I've got déjà vu about this suggestion.

			label border
			`	"The raids have proven that we are not prepared to ensure true security on our borders. In order to address this we want to reactivate the mothballed station of <destination>, for which we have acquired the relevant paperwork and access codes. We have already hired a few mercenaries to help with the defense of our station, mostly local humans with experienced pasts. Once operational, the station should be a major strategic advantage for us, allowing resupply and repairs of ships at a location suitable for facilitating rapid response to detected threats."`

> 			`	He indicates a door to another room, "The Stationkeeper will be out shortly. We've almost completed our final preparations, but we've been held up trying to make do with the firepower that's signed up so far. Judging by your credentials, I suspect that Kelamursh will want to travel with you. You will need to escort them to the station and dock, so they can confirm the access codes function and that there are no critical errors in the initial restoration process, as well as assess if any additional supplies are required."`

Reviewer: ```suggestion
			`	He indicates a door to another room. "The Stationkeeper will be out shortly. We've almost completed our final preparations, but we've been held up trying to make do with the firepower that's signed up so far. Judging by your credentials, I suspect that Kelamursh will want to travel with you. You will need to escort them to the station and dock, so they can confirm the access codes function and that there are no critical errors in the initial restoration process, as well as assess if any additional supplies are required."`
```

Or, if you like commas:
```suggestion
			`	He indicates a door to another room and continues, "The Stationkeeper will be out shortly. We've almost completed our final preparations, but we've been held up trying to make do with the firepower that's signed up so far. Judging by your credentials, I suspect that Kelamursh will want to travel with you. You will need to escort them to the station and dock, so they can confirm the access codes function and that there are no critical errors in the initial restoration process, as well as assess if any additional supplies are required."`
```

		"reputation: Hai Merchant" += 200
		"reputation: Hai Merchant (Human)" += 200
		"reputation: Hai Merchant (Sympathizers)" += 200
		event "hr: sandsnap awoken" 5
		fail "HRCP: Sandsnap Mercenaries"
		# Ideally eventually use: payment ( "armament deterrence" * 18000 - "cargo attractiveness" * 3000 )
		payment 1200000
		conversation
			`As you're re-docking behind the supply convoy, a message pings on your console. It contains a short text from "Heyru" thanking you for your assistance and advising that you've been sent a payment of <payment> for your efforts.`
			`	It seems that the security mechanisms in the docking section have been reactivated, so they no longer need their mercenaries to hang around.`

event "hr: sandsnap awoken"
	planet Sandsnap
		government "Hai Merchant"
		attributes hai station
		landscape land/station35
		music ambient/machinery
		description `This station was recently re-powered by a private military contractor to serve as a resupply and repair base that Hai warships can be stationed at to respond more rapidly to pirates entering the system. There has been a constant presence here since it was set up shortly after the pirate raids on Allhome.`

> 		spaceport `The operating model for the spaceport is evidently based on the military base at Everhope, but the station layout makes restricting portions of the promenade unfeasible. This leaves the dual-level promenade fully open to civilian access, with security restrictions in place at all access points to the higher levels. The floor has had new texturing laid down to prevent slipping, but nearly every other surface appears to be coated in a fine, sticky, metallic powder. Frequently touched surfaces have been polished clean, but any accidental contact with the powder results in leaving fingerprints on any non-metallic surface until thoroughly wiped off.`

Reviewer: ```suggestion
		spaceport `The operating model for the spaceport is evidently based on the military base at Everhope, but Sandsnap's layout makes restricting portions of the promenade unfeasible. While this leaves the dual-level promenade fully open to civilian access, security restrictions have been put in place at all access points to the higher levels.`
		`	The floor has had new texturing laid down to prevent slipping, but nearly every other surface appears to be coated in a fine, sticky, metallic powder. Frequently touched surfaces have been polished clean, but otherwise any accidental contact with the powder results in fingerprints being left on any non-metallic surface until it is thoroughly wiped off.`
```
I understand if changing `station layout` to `Sandsnap's layout` might not be preferable since it's about civilian station layouts, but `Sandsnap's layout` is more clear to me.

	destination "Allhome"

> 	to offer
> 		has "Hai Leaks Response 2: done"
> 		has "Hai Leaks Response 2: aborted"

Reviewer 4: ```suggestion
	to offer
	        or
		        has "Hai Leaks Response 2: done"
		        has "Hai Leaks Response 2: aborted"
```

	source "Hai-home"
	destination "Allhome"
	to offer
		or
			has "Hai Leaks Response 2: done"
			has "Hai Leaks Response 2: aborted"
	on offer
		conversation

> 			`You are directed to an official landing pad on your return to Hai-home.`
> 			`	A functionary leads you to what appears to be a situation room, filled with monitors, holoprojectors, and other equipment. All of the diplomats are here except for Remington, who arrives moments behind you. Sayari gestures for you to take a seat as she switches on a massive holoprojector embedded in the table.`
> 			`	She nods to you and says, "Other plans are proceeding but I asked for you to come back and join the discussion for this. It seems the increase in pirate activity is only worsening over time since the media retraction, and while innocents may be keeping away, that helps us very little."`

Reviewer: ```suggestion
			`You are directed to an official landing pad on your return to Hai-home, and after securing the <ship>, a functionary leads you to what appears to be a situation room, filled with monitors, holoprojectors, and other equipment. All of the diplomats are here except for Remington, who arrives moments behind you.`
			`	Sayari gestures for you to take a seat as she switches on a massive holoprojector embedded in the table. She nods to you and says, "Other plans are proceeding, but I asked for you to come back and join the discussion for this. It seems the increase in pirate activity has only worsened over time since the media retraction, and while innocents may be keeping away, that helps us very little."`
```
Seems weird that the landing pad is singled out in its own line.

`Securing the <ship>` could be improved, I wanted something that could bridge the sentences.

	source "Hai-home"
	destination "Allhome"
	to offer
		or
			has "Hai Leaks Response 2: done"
			has "Hai Leaks Response 2: aborted"
	on offer
		conversation
			`You are directed to an official landing pad on your return to Hai-home, and after securing the <ship>, a functionary leads you to what appears to be a situation room, filled with monitors, holoprojectors, and other equipment. All of the diplomats are here except for Remington, who arrives moments behind you.`
			`	Sayari gestures for you to take a seat as she switches on a massive holoprojector embedded in the table. She nods to you and says, "Other plans are proceeding, but I asked for you to come back and join the discussion for this. It seems the increase in pirate activity has only worsened over time since the media retraction, and while innocents may be keeping away, that helps us very little."`
			choice
				`	"I don't recall there being so many pirates even in human space."`

> 				`	"It is getting out of hand. Where are they all coming from?."`

Reviewer 5: ```suggestion
				`	"It is getting out of hand. Where are they all coming from?"`
```

			`	He checks the date. "Our most optimistic estimates suggest the Republic will be forced to acknowledge the existence of the Hai within three months, or we would lose so much credibility as to compromise electoral stability. The actual time limit is probably sooner."`
			choice
				`	"Would that be so bad?"`

> 			`	"It is always bad to be forced into acknowledging a secret. If we act sooner, we may control the manner of that secret's release," says Xilin. "This pirate situation is accelerating the timeline in an alarming way, we need to get on top of it."`

Reviewer: ```suggestion
			`	"It is always bad to be forced into acknowledging a secret. If we act sooner, we may control the manner of that secret's release," says Xilin. "This pirate situation is accelerating our timeline in an alarming way, and we need to get on top of it."`
```
Perhaps better? It seemed a little clichéish before.

			`	Sayari looks at you. "We are developing and propagating cover stories for some artifacts, and quickly, but for the moment may I ask a favor of the war hero in the room? Please meet up with the extra security forces converging on Allhome and help them rout the pirates back to the other side of the wormhole. It will buy us some breathing room and options."`

> 			`	Even were the entire diplomatic core not looking at you expectantly, it is so hard to say "no" to a Hai and their cute squirrelly features.`

Reviewer: ```suggestion
			`	Even if the combined diplomatic corps of humanity and Hai weren't looking at you expectantly, it is so hard to say "no" to a Hai and their cute squirrelly features.`
```
This last line is still weird to me after these years.
